<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Variant Interpretation COSI">
    <meta name="author" content="Emidio Capriotti">
    <link rel="shortcut icon" href="./favicon.ico" type="image/vnd.microsoft.icon" />

    <title>Variant Interpretation - COSI </title>

    <!-- Bootstrap core CSS -->
    <link href="./css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <link href="../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    -->

    <!-- Custom styles for this template -->
    <link href="./css/jumbotron-narrow-biofold.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
    <script src="../assets/js/ie-emulation-modes-warning.js"></script>
    -->



  </head>

  <body>
    <br>
    <div class="container">

      <div class="jumbotron">
        <table width="100%">
        <tr>
        <td width="80%">
	<div class='biofold' >

        <span style="color:#d97b2d;font-size:150%">VarI</span>
        <span style="color:#d97b2d;font-size:50%">c</span><span style="font-size:50%">ommunity </span>
        <span style="color:#d97b2d;font-size:50%">o</span><span style="font-size:50%">f  </span>
        <span style="color:#d97b2d;font-size:50%">s</span><span style="font-size:50%">pecial </span>
        <span style="color:#d97b2d;font-size:50%">i</span><span style="font-size:50%">nterest </span>

        </div>
        <!--
        <div class='biofold' ><span style="color:#c31e27;">Bio</span><span style="color:#4689a3;">molecules</span> <span style="color:#c31e27;">Fol</span><span style="color:#4689a3;">ding and </span><span style="color:#c31e27;">D</span><span style="color:#4689a3;">isease</span></div><br>
        -->
	</td>
	<td width="20%">
	<div class="right_cell">
        <a href="./index.html">
        <img src="./img/abacov.jpg" width="150px" align="center">
        </a>
        </div>
	</td>
	</tr>
	</table>
  <br>
	<table width="100%">
        <tr>
        <td>
        <div class="form-inline justify-content-center">
        <nav>
          <ul class="nav nav-pills">
            <li role="presentation" ><a href="./index.html">Home</a></li>
            <li role="presentation" class='active'><a href="">Schedule</a></li>
            <li role="presentation" ><a href="./submission.html">Submission</a></li>
            <li role="presentation" ><a href="./publication.html">Publication</a></li>
            <li role="presentation" ><a href="./company.html">Company</a></li>
	    <li role="presentation"><a href="./contact.html">Contact</a></li>
          </ul>
        </nav>
        </div>
        </td>
        </tr>
        </table>
    <br>
  	<br>

     <div class="text">
     <h4 class="title" >VarI COSI 2023</h4>
     </div>

  <div class="text">
    <p class="text">
      <span style="color:#d97b2d;font-weight:400;">What:</span> Variant Interpretation Session at the ISMB/ECCB 2023<br>
      <span style="color:#d97b2d;font-weight:400;">When:</span> 8:30 - 16:30 CEST, 27 July, 2023.</b> <br>
      <span style="color:#d97b2d;font-weight:400;">Where:</span>  Convention Center, Salle Rhone 1 - Lyon, France and Online Meeting.
    </p>

    <table  border="0"  cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" >
      <colgroup>
        <col style="width:20%">
        <col style="width:600%">
        </colgroup>
      <tbody>
<!--
      <tr>

        <td colspan="2"><span><p class="text-small"><br><br><strong>
              <img src="/img/PU.gif" width="12" height="12"> Key Dates
        </strong>
        <br><br></span></td>
      </tr>
      <tr>
      <td><span><p class="text-small"> <strong>January 24, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"> Call for Abstracts Opens <br>&nbsp; </span></td>
      </tr>
      <tr>
      <td><span><p class="text-small"> <strong>April 21, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small">  <strong>Abstracts Submission Deadline</strong> <br>&nbsp; </span></td>
      </tr>

      <tr>
      <td><span><p class="text-small"> <strong>April 25, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"> Late Posters Submissions Open <br>&nbsp; </span></td>
      </tr>

      <tr>
      <td><span><p class="text-small"> <strong>May 12, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"> Talk and/or Poster Acceptance Notifications <br>&nbsp; </span></td>
      </tr>
      <tr>
      <td><span><p class="text-small"> <strong>May 19, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"> <strong>Late Posters Submissions Deadline</strong> <br>&nbsp; </span></td>
      </tr>
      <tr>
      <td><span><p class="text-small"> <strong>May 19, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"> Late Posters Acceptance Notifications <br>&nbsp; </span></td>
      </tr>
      <tr>
      <td><span><p class="text-small"> <strong>June 10, 2022</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"> Presenters have to complete the release forms, register and confirm their participation <br>&nbsp; </span></td>
      </tr>


--->

      <tr>

        <td colspan="2"><span><p class="text-small"><br><br><strong>
              <img src="/img/PU.gif" width="12" height="12"> Session 1
<!-- Tuesday July 12, 2022
Annotation and prediction of structural/functional impacts of genetic variants.
-->


        </strong>
        <br><br></span></td>
      </tr>
      <tr>
        <td><span><p class="text-small"> <strong>8:30 - 8:40</strong><br>&nbsp;</span></td>
        <td><span><p class="text-small"> Opening remarks<br>&nbsp; </span></td>
      </tr>
      <tr>
        <td><span><p class="text-small"><strong>8:40 - 8:50</strong><br>&nbsp;</span></td>
        <td><span><p class="text-small">
        <u>Yazdan Asgari.</u> Paris-Saclay University, Paris-Sud University, Gustave Roussy, Villejuif, France.
          <br><i>
          Developing a new pipeline for exploring pleiotropy of GWAS data at gene-level.
          </i>
      </span></td>
      </tr>

      <tr>
        <td><span><p class="text-small"><strong>8:50 - 9:30</strong><br>&nbsp;<br>&nbsp;</span></td>
        <td><span><p class="text-small"><strong>Keynote talk:</strong>

        <a href="https://www.iarc.who.int/staff_member/matthieu-foll/" target=blank>
          <strong>Matthieu Foll</strong></a>.
        International Agency for Research on Cancer, Lyon, France.
        <br>
        <i>
         Multi-omics characterization of rare heterogeneous tumors.
        </i><br>&nbsp;
        </span></td>
        </tr>

              <tr>
              <td><span><p class="text-small"><strong>9:30 - 10:00</strong><br>&nbsp;
              </span></td><td><span><p class="text-small"><b>Coffee Break</b><br>&nbsp;</span></td>
              </tr>

      <tr>
        <td><span><p class="text-small"><strong>10:00 - 10:10</strong><br>&nbsp
        </span></td>
        <td><span><p class="text-small"><span><p class="text-small">
        <u>Vaishali Waman.</u> University College London, London, United Kingdom.  <br><i>
        Predicting human and viral protein variants affecting COVID-19 susceptibility.</i>
        </span></td>
      </tr>
      <tr>
        <td><span><p class="text-small"><strong>10:10 - 10:30</strong><br>&nbsp
        </span></td>
        <td><span><p class="text-small"><span><p class="text-small">
        <u>Kivilcim Ozturk.</u> University of California San Diego, La Jolla (CA), USA.  <br><i>
        Interface-guided phenotyping of coding variants.</i>
        </span></td>
      </tr>

      <tr>
        <td><span><p class="text-small"><strong>10:30 - 10:40</strong><br>&nbsp<br>&nbsp
        </span></td>
        <td><span><p class="text-small"><span><p class="text-small">
        <u>Swatantra Pradhan.</u> Tata Consultancy Services, India.  <br><i>
        An augmented transformer model trained on family specific variant data leads to improved prediction of variants<br>
        of uncertain significance.</i>
        </span></td>
      </tr>

      <tr>
        <td><span><p class="text-small"><strong>10:40 - 10:50</strong><br>&nbsp;<br>&nbsp;</span></td>
        <td><span><p class="text-small">
        <u>Nigreisy Montalvo.</u> Imagine Institute, Paris, France.
          <br><i>
          Pathogenicity scoring of genetic variants through federated learning across independent institutions reaches
          <br>comparable or superior performance than the centralized-data model counterparts.
          </i>
      </span></td>
      </tr>
      <tr>
        <td><span><p class="text-small"><strong>10:50 - 11:00</strong><br>&nbsp
        </span></td>
        <td><span><p class="text-small"><span><p class="text-small">
        <u>Julian Gough.</u> MRC Laboratory of Molecular Biology, United Kingdom.<br><i>
        Hypothesis-free phenotype prediction within a genetics-first framework.</i>
        </span></td>
      </tr>

      <tr>
        <td><span><p class="text-small"><strong>11:00 - 11:20</strong><br>&nbsp
        </span></td>
        <td><span><p class="text-small"><span><p class="text-small">
        <u>Alexander Schoenhuth.</u> Bielefeld University, Germany.<br><i>
        Predicting the prevalence of complex genetic diseases from individual
        genotype profiles using capsule networks.</i>
        </span></td>
      </tr>

      <tr>
        <td><span><p class="text-small"><strong>11:20 - 12:00</strong><br>&nbsp;<br>&nbsp;</span></td>
        <td><span><p class="text-small"><strong>Keynote talk:</strong>

        <a href="https://menchelab.com/joerg-menche" target=blank>
          <strong>Jörg Menche</strong></a>.
        University of Vienna, Austria
        <br>
        <i>
         Network Medicine – From protein-protein to human-machine interactions.
        </i>
        <br>&nbsp;</span></td>
      </tr>
<!--
      <tr>
      <td><span><p class="text-small"><strong>12:20 - 12:40</strong><br>&nbsp;</span></td>
      <td><span><p class="text-small"><b>Coffee Break</b><br>&nbsp;</span></td>
      </tr>
-->

         </tr>

         <tr>
         <td><span><p class="text-small"><strong>12:00 - 13:20</strong><br>&nbsp;</span></td>
         <td><span><p class="text-small"><b>Lunch break</b><br>&nbsp;</span></td>
         </tr>

<tr>
        <td colspan="2"><span><p class="text-small"><strong><br />

              <img src="/img/PU.gif" alt="" width="12" height="12" />
              Session 2

<!--
- Tuesday July 12, 2022
Genetic variants as effectors of change: disease and evolution. <br />
-->



        <br /><br />
        </strong></span></td>
        </tr>


        <tr>
                <td><span><p class="text-small"><strong>13:20 - 13:40</strong><br>&nbsp;</span></td>
                <td><span><p class="text-small">

                <u>Rongting Huang</u>. The University of Hong Kong, Hong Kong.
                <br><i>
                XClone: detection of allele-specific subclonal copy number
                variations from single-cell transcriptomic data.</i>
              </span></td>
        </tr>

        <tr>
                <td><span><p class="text-small"><strong>13:40 - 14:00</strong><br>&nbsp;<br>&nbsp;</span></td>
                <td><span><p class="text-small">

                <u>David Wang</u>. University of Pennsylvania, Philadelphia (PA) USA.
                <br><i>
              Comprehensive Identification and Characterization of Splicing Associated
              Variants with Coverage Aware<br>
              Statistical Models.</i>
              </span></td>
        </tr>

        <tr>
                <td><span><p class="text-small"><strong>14:00 - 14:10</strong><br>&nbsp;<br>&nbsp;</span></td>
                <td><span><p class="text-small">

                <u>Nina Baumgarten</u>. Goethe University, Frankfurt am Main, Germany.
                <br><i>
                  A statistical approach to identify regulatory DNA variations
                  combined with epigenomics data reveals novel<br>
                  non-coding disease genes.</i>
              </span></td>
        </tr>

        <tr>
                <td><span><p class="text-small"><strong>14:10 - 14:20</strong><br>&nbsp;</span></td>
                <td><span><p class="text-small">

                <u>Eleni Giannoulatou</u>. Victor Chang Cardiac Research Institute, Sydney (NSW), Australia.
                <br><i>
                  SpliceSM: machine learning discovery of splice-altering
                  variants using susceptibility maps.</i>
              </span></td>
        </tr>

        <tr>
          <td><span><p class="text-small"><strong>14:20 - 15:00</strong><br>&nbsp;</span></td>
        <td><span><p class="text-small"><strong>Keynote talk:</strong>

      <a href="https://www.well.ox.ac.uk/people/nicola-whiffin"target=blank><strong>Nicky Whiffin</strong></a>.
        University of Oxford, Oxford, United Kingdom.
        <br>
         <i>
        Strategies to annotate and interpret non-coding variants in rare disease.
         </i>

        </span></td>
      </tr>

<!--
<tr>
             <td><span><p class="text-small"><strong>15:10 - 15:30</strong><br>&nbsp;</span></td>
             <td><span><p class="text-small"><b>VarI sponsor:</b>

            <b><a href="https://www.variantyx.com/" target=blank>Shane Hussey - Variantyx</a></b>.
            <br>
             <i>Validation of WGS for clinical use down to 50 ng of DNA using tagmentation PCR-Free protocol.
     </i>
             </span></td>
           </tr>
-->


	     <tr>
             <td><span><p class="text-small"><br><strong>15:00 - 15:30</strong><br>&nbsp;</span></td>
             <td><span><p class="text-small"><br><b>Coffee Break</b><br>&nbsp;</span></td>
             </tr>


         <tr>
                <td><span ><p class="text-small"><strong>15:30 - 15:50</strong><br>&nbsp;<br>&nbsp;</span></td>
                <td><span ><p class="text-small"><strong>Proceedings Talk:</strong>

                          <u>Yasser Mohseni Behbahani</u>. Sorbonne Université,
                          Paris, France.
                <br><i>
                  Deep Local Analysis deconstructs protein - protein interfaces and accurately estimates binding affinity<br>changes upon mutation.
                </span></td>
              </tr>



   <tr>
      <td><span><p class="text-small"><strong>15:50 - 16:10</strong><br>&nbsp;<br>&nbsp;</span></td>
        <td><span><p class="text-small">
          <u>Jayoung Ryu</u>. Harvard Medical School, Boston (MA), USA.
          <br>
          <i>
          Modeling endogenous editing outcome of base editor
          reporter screens with CRISPR-Bean discovers causal<br>
          variants for cellular LDL uptake.
        </i>

        <tr>
           <td><span><p class="text-small"><strong>16:10 - 16:20</strong><br>&nbsp;<br>&nbsp;</span></td>
             <td><span><p class="text-small">

               <u>Daniele Raimondi</u>. KU Leuven, Leuven, Belgium.
               <br>
               <i>
                 From genotype to phenotype in Arabidopsis thaliana:
                 in-silico genome interpretation predicts 288 phenotypes<br>
                 from sequencing data.
             </i>


             </span></td>
           </tr>


        </span></td>
      </tr>
      <tr>


      <tr>
        <td><span><p class="text-small"><br><strong>16:20 - 16:30</strong></span></td>
        <td><span><p class="text-small"><br>Closing remarks.</span></td>
      </tr>



    </table>
    </p>
    <br>
    <br>
    <!---
    <div class="text">
    <h4 class="title" >Meeting program</h4>
    </div>
    <p class="text">
    TBA
    <p>
    <table border=0 width="100%">

    <tbody>
    <tr>
    <td  align=center>
    </td>
	  </tr>
	  <tbody>
	  </table>


    </p>

    <br>
    <br>
    -->
    <div class='footer-ok'><div>

      </div>
    </div> <!-- /container -->


    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <script src="../assets/js/ie10-viewport-bug-workaround.js"></script>
    -->
  </body>
</html>
