<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Vari special interest group</title>
<style type="text/css">

body {
	background-color: #FE9901;
}
#dropmenudiv{
position:absolute;
border:1px WHITE;
border-bottom-width: 0;
font:normal 14px Verdana, Arial, Helvetica, sans-serif;
line-height:16px;
z-index:100;
}

#dropmenudiv a{
width: 220px;
display: block;
text-indent: 3px;
border-bottom: 1px WHITE;
padding: 1px 0;
text-decoration: none;
font-weight: bold;
}

#dropmenudiv a:hover{ /*hover background color*/
color: #79a631;
background-color: #ffffff;
}

</style>
<script type="text/javascript">
//Contents for menu 2, and so on
var menu2=new Array()
menu2[0]='<a href="topics.html">Topics</a>'
menu2[1]='<a href="kd.html">Key dates</a>'
menu2[2]='<a href="schedu.html">Schedule</a>'

var menu3=new Array()
menu3[0]='<a href="abstract.html">Talks and Posters</a>'
menu3[1]='<a href="company.html">Company Involvement</a>'
menu3[2]='<a href="special.html">Special Issue</a>'

var menu4=new Array()
menu4[0]='<a href="people.html">People</a>'
menu4[1]='<a href="conta.html">Contact</a>'

		
var menuwidth='165px' //default menu width
var menubgcolor='#FFFFFF'  //menu bgcolor
var disappeardelay=250  //menu disappear speed onMouseout (in miliseconds)
var hidemenu_onclick="yes" //hide menu when user clicks within menu?

/////No further editting needed

var ie4=document.all
var ns6=document.getElementById&&!document.all

if (ie4||ns6)
document.write('<div id="dropmenudiv" style="visibility:hidden;width:'+menuwidth+';background-color:'+menubgcolor+'" onMouseover="clearhidemenu()" onMouseout="dynamichide(event)"></div>')

function getposOffset(what, offsettype){
var totaloffset=(offsettype=="left")? what.offsetLeft : what.offsetTop;
var parentEl=what.offsetParent;
while (parentEl!=null){
totaloffset=(offsettype=="left")? totaloffset+parentEl.offsetLeft : totaloffset+parentEl.offsetTop;
parentEl=parentEl.offsetParent;
}
return totaloffset;
}

function showhide(obj, e, visible, hidden, menuwidth){
if (ie4||ns6)
dropmenuobj.style.left=dropmenuobj.style.top=-500
if (menuwidth!=""){
dropmenuobj.widthobj=dropmenuobj.style
dropmenuobj.widthobj.width=menuwidth
}
if (e.type=="click" && obj.visibility==hidden || e.type=="mouseover")
obj.visibility=visible
else if (e.type=="click")
obj.visibility=hidden
}

function iecompattest(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function clearbrowseredge(obj, whichedge){
var edgeoffset=0
if (whichedge=="rightedge"){
var windowedge=ie4 && !window.opera? iecompattest().scrollLeft+iecompattest().clientWidth-15 : window.pageXOffset+window.innerWidth-15
dropmenuobj.contentmeasure=dropmenuobj.offsetWidth
if (windowedge-dropmenuobj.x < dropmenuobj.contentmeasure)
edgeoffset=dropmenuobj.contentmeasure-obj.offsetWidth
}
else{
var topedge=ie4 && !window.opera? iecompattest().scrollTop : window.pageYOffset
var windowedge=ie4 && !window.opera? iecompattest().scrollTop+iecompattest().clientHeight-15 : window.pageYOffset+window.innerHeight-18
dropmenuobj.contentmeasure=dropmenuobj.offsetHeight
if (windowedge-dropmenuobj.y < dropmenuobj.contentmeasure){ //move up?
edgeoffset=dropmenuobj.contentmeasure+obj.offsetHeight
if ((dropmenuobj.y-topedge)<dropmenuobj.contentmeasure) //up no good either?
edgeoffset=dropmenuobj.y+obj.offsetHeight-topedge
}
}
return edgeoffset
}

function populatemenu(what){
if (ie4||ns6)
dropmenuobj.innerHTML=what.join("")
}

function dropdownmenu(obj, e, menucontents, menuwidth){
if (window.event) event.cancelBubble=true
else if (e.stopPropagation) e.stopPropagation()
clearhidemenu()
dropmenuobj=document.getElementById? document.getElementById("dropmenudiv") : dropmenudiv
populatemenu(menucontents)

if (ie4||ns6){
showhide(dropmenuobj.style, e, "visible", "hidden", menuwidth)
dropmenuobj.x=getposOffset(obj, "left")
dropmenuobj.y=getposOffset(obj, "top")
dropmenuobj.style.left=dropmenuobj.x-clearbrowseredge(obj, "rightedge")+"px"
dropmenuobj.style.top=dropmenuobj.y-clearbrowseredge(obj, "bottomedge")+obj.offsetHeight+"px"
}

return clickreturnvalue()
}

function clickreturnvalue(){
if (ie4||ns6) return false
else return true
}

function contains_ns6(a, b) {
while (b.parentNode)
if ((b = b.parentNode) == a)
return true;
return false;
}

function dynamichide(e){
if (ie4&&!dropmenuobj.contains(e.toElement))
delayhidemenu()
else if (ns6&&e.currentTarget!= e.relatedTarget&& !contains_ns6(e.currentTarget, e.relatedTarget))
delayhidemenu()
}

function hidemenu(e){
if (typeof dropmenuobj!="undefined"){
if (ie4||ns6)
dropmenuobj.style.visibility="hidden"
}
}

function delayhidemenu(){
if (ie4||ns6)
delayhide=setTimeout("hidemenu()",disappeardelay)
}

function clearhidemenu(){
if (typeof delayhide!="undefined")
clearTimeout(delayhide)
}

if (hidemenu_onclick=="yes")
document.onclick=hidemenu

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
</script>
<link rel="stylesheet" href="css/style.css" 
type="text/css" title="Snp sig">
</head>
<body onload="MM_preloadImages('/img/bottoni-home2.gif')">
<div id="main">
<br />
<br />
<br />
<table width="880" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" class="style1">
  <tr>
    <td width="39" height="39" align="top"><img src="/img/angolo.jpg" alt="" width="39" height="39" align="top" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="/img/angolo-alt-ds.jpg" width="39" height="39" align="top" /></td>
  </tr>
  <tr>
    <td width="39">&nbsp;</td>
    <td><div id="content"><table border="0">
      <tr>
        <td colspan="3"><img src="/img/vari.gif" width="524" height="110" /></td>
        <td><div align="right"><img src="/img/abacov.jpg" width="162" height="110" /></div></td>
      </tr>
     <tr>
        <td ><br >
          </td>
        </tr>
      <tr>
        <td><div id="menu"><a href="index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image7','','/img/bottoni-home2.gif',1)"><img src="/img/bottoni-home.gif" name="Image7" width="45" height="22" border="0" id="Image7" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu2, '165px')" onMouseout="delayhidemenu()"><img src="/img/bottoni-prog.gif" width="78" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu3, '165px')" onMouseout="delayhidemenu()"><img src="/img/bottoni-sub.gif" width="113" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu4, '165px')" onMouseout="delayhidemenu()"><img src="/img/bottoni-organizz.gif" width="138" height="22" border="0" /></a></div></td>
      </tr>
      <tr>
        <td colspan="4"> <p class="textzone">
        <br />
        <br />
	As for the previous editions, the VarI-COSI organizers will invite the authors of <strong>the presented papers</strong> and <strong> strong posters</strong>
        to submit their work for the next special issue publication.<br><br>
	
	The <b><a href="http://varisig.biofold.org/2018/" target=blank>VarI-COSI 2018</a></b> Special Issue 
	has been published on BMC Genomics and Medical Genomics and Bioinformatics. The accepted papers of the special issue 
	are available at the
        <a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-20-supplement-8"
        target=blank>BMC Genomics</a>,
	
	<a href="https://bmcmedgenomics.biomedcentral.com/articles/supplements/"
	target=blank>BMC Medical Genomics</a>
        and
        <a href="https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-20-supplement-14"
        target=blank>BMC Bioinformatics</a> web pages.

	<br><br>
	The list of the selected manuscript is reported below. 
	<br><br>

	<ul>
	<li>
	Bromberg Y, Capriotti E, Carter H. VarI-COSI 2018: A forum for research advances in variant interpretation and diagnostics. BMC Genomics. 2019, 20 (Suppl 8): 550.	
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307380">PMID:31307380</a>.
        </li><br>

	<li>
        Field M, Burgio G, Chuah A, Al Shekaili J, Simon F, Cook M, Andrews DT. Recurrent miscalling of missense variation from short-read genome sequence data information. BMC Genomics. 2019;20(Suppl 8):546.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307400">PMID:31307400</a>.
        </li><br>
	
	<li>
	Polte C, Wedemeyer D, Wagner J, Oliver K, Bijvelds M, Mahoney J, de Jonge H, Sorscher E, Ignatova Z. Assessing cell-specific effects of genetic variation using tRNA microarrays. BMC Genomics. 2019;20(Suppl 8):549.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307398">PMID:31307398</a>.
	</li><br>

	<li>
	Orioli T, Vihinen M. Benchmarking membrane proteins: subcellular localization and variant tolerance predictors. BMC Genomics. 2019;20(Suppl 8):547.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307390">PMID:31307390</a>.
	</li><br>

	<li>
	Neerman N, Faust G, Meeks N, Modai S, Kalfon L, Falik-Zaccai T, Kaplun A. A clinically validated whole genome pipeline for structural variant detection and analysis. BMC Genomics. 2019;20(Suppl 8):545.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307387">PMID:31307387</a>.
	</li><br>	

	<li>
	Pucci F, Rooman M. More insights into the relation between DNA ionization potentials, single base substitutions and pathogenic mutations. BMC Genomics. 2019;20(Suppl 8):551.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307386">PMID:31307386</a>.
	</li><br>	

	<li>
        Babbi G, Martelli P, Casadio R. PhenPath: a tool for characterizing biological functions underlying different phenotypes. BMC Genomics. 2019;20(Suppl 8):548.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307376">PMID:31307376</a>.
        </li><br>

        <li>
	Ascolani G, Lio’ P. Modeling breast cancer progression to bone: how driver mutation order and metabolism matter. BMC Med Genet. 2019;12(Suppl 6):S1.
	<a href="https://bmcmedgenomics.biomedcentral.com/track/pdf">PMID:TBA</a>
	</li><br>

	<li>
	Glicksberg B, Amadori L, Akers N, Sukhavasi K, Franzén O, Li L, Belbin G, Ayers K, Shameer K, Badgeley M, et al. Integrative analysis of loss-of-function variants in clinical and genomic data reveals novel genes associated with cardiovascular traits. BMC Med Genet. 2019;12(Suppl 6):S2.
	<a href="https://bmcmedgenomics.biomedcentral.com/track/pdf">PMID:TBA</a>
	</li><br>

	<li>
	Simpson C, Zhang B, Hornbeck P, Gnad F. Systematic analysis of the intersection of disease mutations with protein modifications. BMC Med Genet. 2019;12(Suppl 6):S3.
	<a href="https://bmcmedgenomics.biomedcentral.com/track/pdf">PMID:TBA</a>
	</li><br>

	<li>
	Castro A, Ozturk K, Pyke R, Xian S, Zanetti M, Carter H. Elevated neoantigen levels in tumors with somatic mutations to B2M and HLA. BMC Med Genet. 2019;12(Suppl 6):S4.	
	<a href="https://bmcmedgenomics.biomedcentral.com/track/pdf">PMID:TBA</a>
	</li><br>	

	<li>
Montanucci L, Capriotti E, Frank Y, Ben-Tal N, Fariselli P. DDGun: an untrained method for the prediction of protein stability changes upon single and multiple point variations. BMC Bioinformatics. 2019;20(Suppl 14):335.
	<a href="https://www.ncbi.nlm.nih.gov/pubmed/31307380">PMID:31307380</a>
	</li><br>

	</ul>


        The BMC special issue of the <b><a href="http://varisig.biofold.org/2015" target=blank>VarI-COSI 2016</a></b> has been published on <b><a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-18-supplement-5" target=blank>BMC Genomics</a></b> and <b><a href="https://bmcbiophys.biomedcentral.com/articles/supplements/volume-10-supplement-1" target=blank>BMC Biophysics</a></b>.
	<br><br>
        The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2015" target=blank>VarI-SIG 2015</a></b> is available at the following
        <b><a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-17-supplement-2" target=blank>web page</a></b>.
        <br>
	<br>
	The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2014" target=blank>VarI-SIG 2014</a></b> is available at the following
        <b><a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-16-supplement-8" target=blank>web page</a></b>.
        <br><br>
	The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2013" target=blank>SNP-SIG 2013</a></b> is available at the following
        <b><a href="http://www.biomedcentral.com/bmcgenomics/supplements/15/S4" target=blank>web page</a></b>.
        <br><br>
        The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2012" target=blank>SNP-SIG 2012</a></b> is available at the following
	<b><a href="http://www.biomedcentral.com/bmcgenomics/supplements/14/S3" target=blank>web page</a></b>.
        <br><br>
        The BMC Genomics special issue of the <b><a href="http://varisig.biofold.org/2011" target=blank>SNP-SIG 2011</a></b> is available at the following
        <b><a href="http://www.biomedcentral.com/bmcgenomics/supplements/13/S4" target=blank>web page</a></b>.
        </p>
          <p class="textzone"><br />
            </p></td>
      </tr>
      <tr>
        <td colspan="4" class="footer" ></td>
      </tr>
    </table>
    </div>
    <!--End Content-->
      </td>
    <td width="39">&nbsp;</td>
  </tr>
  
  <tr>
    <td width="39" height="39"><img src="/img/angolo-abasso-sin.jpg" width="39" height="39" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="/img/angolo-abasso-ds.jpg" width="39" height="39" /></td>
  </tr>
</table>
<br>
<br>
</div>
</body>
</html>
