<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Vari special interest group</title>
<style type="text/css">

body {
	background-color: #FE9901;
}
#dropmenudiv{
position:absolute;
border:1px WHITE;
border-bottom-width: 0;
font:normal 14px Verdana, Arial, Helvetica, sans-serif;
line-height:16px;
z-index:100;
}

#dropmenudiv a{
width: 220px;
display: block;
text-indent: 3px;
border-bottom: 1px WHITE;
padding: 1px 0;
text-decoration: none;
font-weight: bold;
}

#dropmenudiv a:hover{ /*hover background color*/
color: #79a631;
background-color: #ffffff;
}

</style>
<script type="text/javascript">
//Contents for menu 2, and so on
var menu2=new Array()
menu2[0]='<a href="topics.html">Topics</a>'
menu2[1]='<a href="kd.html">Key dates</a>'
menu2[2]='<a href="schedu.html">Schedule</a>'

var menu3=new Array()
menu3[0]='<a href="abstract.html">Talks and Posters</a>'
menu3[1]='<a href="company.html">Company Involvement</a>'
menu3[2]='<a href="special.html">Special Issue</a>'

var menu4=new Array()
menu4[0]='<a href="people.html">People</a>'
menu4[1]='<a href="conta.html">Contact</a>'

		
var menuwidth='165px' //default menu width
var menubgcolor='#FFFFFF'  //menu bgcolor
var disappeardelay=250  //menu disappear speed onMouseout (in miliseconds)
var hidemenu_onclick="yes" //hide menu when user clicks within menu?

/////No further editting needed

var ie4=document.all
var ns6=document.getElementById&&!document.all

if (ie4||ns6)
document.write('<div id="dropmenudiv" style="visibility:hidden;width:'+menuwidth+';background-color:'+menubgcolor+'" onMouseover="clearhidemenu()" onMouseout="dynamichide(event)"></div>')

function getposOffset(what, offsettype){
var totaloffset=(offsettype=="left")? what.offsetLeft : what.offsetTop;
var parentEl=what.offsetParent;
while (parentEl!=null){
totaloffset=(offsettype=="left")? totaloffset+parentEl.offsetLeft : totaloffset+parentEl.offsetTop;
parentEl=parentEl.offsetParent;
}
return totaloffset;
}

function showhide(obj, e, visible, hidden, menuwidth){
if (ie4||ns6)
dropmenuobj.style.left=dropmenuobj.style.top=-500
if (menuwidth!=""){
dropmenuobj.widthobj=dropmenuobj.style
dropmenuobj.widthobj.width=menuwidth
}
if (e.type=="click" && obj.visibility==hidden || e.type=="mouseover")
obj.visibility=visible
else if (e.type=="click")
obj.visibility=hidden
}

function iecompattest(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function clearbrowseredge(obj, whichedge){
var edgeoffset=0
if (whichedge=="rightedge"){
var windowedge=ie4 && !window.opera? iecompattest().scrollLeft+iecompattest().clientWidth-15 : window.pageXOffset+window.innerWidth-15
dropmenuobj.contentmeasure=dropmenuobj.offsetWidth
if (windowedge-dropmenuobj.x < dropmenuobj.contentmeasure)
edgeoffset=dropmenuobj.contentmeasure-obj.offsetWidth
}
else{
var topedge=ie4 && !window.opera? iecompattest().scrollTop : window.pageYOffset
var windowedge=ie4 && !window.opera? iecompattest().scrollTop+iecompattest().clientHeight-15 : window.pageYOffset+window.innerHeight-18
dropmenuobj.contentmeasure=dropmenuobj.offsetHeight
if (windowedge-dropmenuobj.y < dropmenuobj.contentmeasure){ //move up?
edgeoffset=dropmenuobj.contentmeasure+obj.offsetHeight
if ((dropmenuobj.y-topedge)<dropmenuobj.contentmeasure) //up no good either?
edgeoffset=dropmenuobj.y+obj.offsetHeight-topedge
}
}
return edgeoffset
}

function populatemenu(what){
if (ie4||ns6)
dropmenuobj.innerHTML=what.join("")
}

function dropdownmenu(obj, e, menucontents, menuwidth){
if (window.event) event.cancelBubble=true
else if (e.stopPropagation) e.stopPropagation()
clearhidemenu()
dropmenuobj=document.getElementById? document.getElementById("dropmenudiv") : dropmenudiv
populatemenu(menucontents)

if (ie4||ns6){
showhide(dropmenuobj.style, e, "visible", "hidden", menuwidth)
dropmenuobj.x=getposOffset(obj, "left")
dropmenuobj.y=getposOffset(obj, "top")
dropmenuobj.style.left=dropmenuobj.x-clearbrowseredge(obj, "rightedge")+"px"
dropmenuobj.style.top=dropmenuobj.y-clearbrowseredge(obj, "bottomedge")+obj.offsetHeight+"px"
}

return clickreturnvalue()
}

function clickreturnvalue(){
if (ie4||ns6) return false
else return true
}

function contains_ns6(a, b) {
while (b.parentNode)
if ((b = b.parentNode) == a)
return true;
return false;
}

function dynamichide(e){
if (ie4&&!dropmenuobj.contains(e.toElement))
delayhidemenu()
else if (ns6&&e.currentTarget!= e.relatedTarget&& !contains_ns6(e.currentTarget, e.relatedTarget))
delayhidemenu()
}

function hidemenu(e){
if (typeof dropmenuobj!="undefined"){
if (ie4||ns6)
dropmenuobj.style.visibility="hidden"
}
}

function delayhidemenu(){
if (ie4||ns6)
delayhide=setTimeout("hidemenu()",disappeardelay)
}

function clearhidemenu(){
if (typeof delayhide!="undefined")
clearTimeout(delayhide)
}

if (hidemenu_onclick=="yes")
document.onclick=hidemenu

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
</script>
<link rel="stylesheet" href="css/style.css" 
type="text/css" title="Snp sig">
</head>
<body onload="MM_preloadImages('img/bottoni-home2.gif')">
<div id="main">
<br />
<br />
<br />
<table width="880" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" class="style1">
  <tr>
    <td width="39" height="39" align="top"><img src="img/angolo.jpg" alt="" width="39" height="39" align="top" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-alt-ds.jpg" width="39" height="39" align="top" /></td>
  </tr>
  <tr>
    <td width="39">&nbsp;</td>
    <td><div id="content"><table border="0">
      <tr>
        <td colspan="3"><img src="img/vari.gif" width="524" height="110" /></td>
        <td><div align="right"><img src="img/abacov.jpg" width="162" height="110" /></div></td>
      </tr>
     <tr>
        <td ><br >
          </td>
        </tr>
      <tr>
        <td><div id="menu"><a href="index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image7','','img/bottoni-home2.gif',1)"><img src="img/bottoni-home.gif" name="Image7" width="45" height="22" border="0" id="Image7" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu2, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-prog.gif" width="78" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu3, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-sub.gif" width="113" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu4, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-organizz.gif" width="138" height="22" border="0" /></a></div></td>
      </tr>
      <tr>
        <td colspan="4"> <p class="textzone">
        <br />
        <br />
	As for the previous editions, the VarI-COSI organizers will invite the authors of <strong>the presented papers</strong> and <strong> strong posters</strong>
        to submit their work for the next special issue publication.<br><br>
	
	The <b><a href="http://varisig.biofold.org/2016/" target=blank>VarI-SIG 2016</a></b> Special Issue 
	has been published on BMC Genomics and BMC Biophysics. The accepted papers of the special issue 
	are available at BMC Genomics 
        <a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-18-supplement-5"
        target=blank>web page</a> 
	and BMC Biophysics 
	<a href="https://bmcbiophys.biomedcentral.com/articles/supplements/volume-10-supplement-1"
	target=blank>web page</a>. 

	<br><br>
	The list of the selected manuscript is reported below. 
	<br><br>

	<li>	
	H. Billur Engin, Daniel Carlin, Dexter Pratt and Hannah Carter. Modeling of RAS complexes supports roles in cancer for less studied partners. BMC Biophysics 2017, 10 (Suppl 1):5.
	<a href="https://bmcbiophys.biomedcentral.com/track/pdf/10.1186/s13628-017-0037-6">PMID:TBA</a>
	</li><br>
        <li>
	Elena Álvarez de la Campa, Natàlia Padilla and Xavier de la Cruz. Development of pathogenicity predictors specific for variants that do not comply with clinical guidelines for the use of computational evidence. BMC Genomics 2017, 18 (Suppl 5):569.
	<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-017-3914-0">PMID:TBA</a>
        </li><br>
	<li>
	Andreas Krämer, Sohela Shah, Robert Anthony Rebres, Susan Tang and Daniel Rene Richards. Leveraging network analytics to infer patient syndrome and identify causal genes in rare disease cases. BMC Genomics 2017, 18 (Suppl 5):551.
	<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-017-3910-4">PMID:TBA</a>
	</li><br>
	<li>
	Giulia Babbi, Pier Luigi Martelli, Giuseppe Profiti, Samuele Bovo, Castrense Savojardo and Rita Casadio. eDGAR: a database of Disease-Gene Associations with annotated Relationships among genes. BMC Genomics 2017, 18 (Suppl 5):554.
	<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-017-3911-3">PMID:TBA</a>
	</li><br>
	<li>
	Morena Pappalardo, Francesca Collu, James Macpherson, Martin Michaelis, Franca Fraternali and Mark N. Wass. Investigating Ebola virus pathogenicity using molecular dynamics. BMC Genomics 2017, 18 (Suppl 5):566.
	<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-017-3912-2">PMID:TBA</a>
	</li><br>
	<li>
	Henry J. Martell, Kathie A. Wong, Juan F. Martin, Ziyan Kassam, Kay Thomas and Mark N. Wass. Associating mutations causing cystinuria with disease severity with the aim of providing precision medicine. BMC Genomics 2017, 18 (Suppl 5):550.
	<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-017-3913-1">PMID:TBA</a>
	</li><br>
<!--

		Pons T, Vazquez M, Matey-Hernandez ML, Brunak S, Valencia A and Izarzugaza JMG: KinMutRF: a random forest classifier of sequence variants in the human protein kinase superfamily. BMC Genomics 2016 17(Suppl 2):396. 
		<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2723-1?site=bmcgenomics.biomedcentral.com"> PMID:TBA</a>.</li><br>
	<li>	
		Kaplun A, Krull M, Lakshman K, Matys V,Lewicki B,  Jennifer D. Hogan JD: Establishing and validating regulatory regions for variant annotation and expression analysis. BMC Genomics 2016 17(Suppl 2):393. 
		<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2724-0?site=bmcgenomics.biomedcentral.com"> PMID:TBA</a>.</li><br>
	<li>	
		Ayers KL, Mirshahi UL, Wardeh AH, Murray MF, Hao K, Glicksberg BS, Li S, Carey DJ, Chen R: A loss of function variant in CASP7 protects against Alzheimer’s disease in homozygous APOE ε4 allele carriers. BMC Genomics 2016 17(Suppl 2):445. 
		<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2725-z?site=bmcgenomics.biomedcentral.com"> PMID:TBA</a>.</li><br>
	<li>
		Martelli PL, Fariselli P, Savojardo C, Babbi G, Aggazio F, Casadio R: Large scale analysis of protein stability in OMIM disease related human protein variants. BMC Genomics 2016 17(Suppl 2):397. 
		<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2726-y?site=bmcgenomics.biomedcentral.com"> PMID:TBA</a>.</li><br>
	<li>	
		Chen T, Wang Z, Zhou W, Chong Z, Meric-Bernstam F, Mills GB, Chen K: Hotspot mutations delineating diverse mutational signatures and biological utilities across cancer types. BMC Genomics 2016 17(Suppl 2):394. 
		<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2727-x?site=bmcgenomics.biomedcentral.com"> PMID:TBA</a></li><br>
	<li>
		Vorontsov IE, Khimulya G, Lukianova EN, Nikolaeva DD, Eliseeva IA, Kulakovskiy IV, Makeev VJ: Negative selection maintains transcription factor binding motifs in human cancer. BMC Genomics 2016 17(Suppl 2):395. 
		<a href="https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-016-2728-9?site=bmcgenomics.biomedcentral.com"> PMID:TBA</a>.
		</li><br>
	</ul>
-->
	<br><br>
        The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2015" target=blank>VarI-SIG 2015</a></b> is available at the following
        <b><a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-17-supplement-2" target=blank>web page</a></b>.
        <br>
	<br>
	The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2014" target=blank>VarI-SIG 2014</a></b> is available at the following
        <b><a href="https://bmcgenomics.biomedcentral.com/articles/supplements/volume-16-supplement-8" target=blank>web page</a></b>.
        <br><br>
	The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2013" target=blank>SNP-SIG 2013</a></b> is available at the following
        <b><a href="http://www.biomedcentral.com/bmcgenomics/supplements/15/S4" target=blank>web page</a></b>.
        <br><br>
        The BMC Gemomics special issue of the <b><a href="http://varisig.biofold.org/2012" target=blank>SNP-SIG 2012</a></b> is available at the following
	<b><a href="http://www.biomedcentral.com/bmcgenomics/supplements/14/S3" target=blank>web page</a></b>.
        <br><br>
        The BMC Genomics special issue of the <b><a href="http://varisig.biofold.org/2011" target=blank>SNP-SIG 2011</a></b> is available at the following
        <b><a href="http://www.biomedcentral.com/bmcgenomics/supplements/13/S4" target=blank>web page</a></b>.
        </p>
          <p class="textzone"><br />
            </p></td>
      </tr>
      <tr>
        <td colspan="4" class="footer" ></td>
      </tr>
    </table>
    </div>
    <!--End Content-->
      </td>
    <td width="39">&nbsp;</td>
  </tr>
  
  <tr>
    <td width="39" height="39"><img src="img/angolo-abasso-sin.jpg" width="39" height="39" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-abasso-ds.jpg" width="39" height="39" /></td>
  </tr>
</table>
<br>
<br>
</div>
</body>
</html>
