<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Variant Interpretation COSI">
    <meta name="author" content="Emidio Capriotti">
    <link rel="shortcut icon" href="./favicon.ico" type="image/vnd.microsoft.icon" />

    <title>Variant Interpretation - COSI </title>

    <!-- Bootstrap core CSS -->
    <link href="./css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <link href="../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    -->

    <!-- Custom styles for this template -->
    <link href="./css/jumbotron-narrow-biofold.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
    <script src="../assets/js/ie-emulation-modes-warning.js"></script>
    -->



  </head>

  <body>
    <br>
    <div class="container">

      <div class="jumbotron">
        <table width="100%">
        <tr>
        <td width="80%">
	<div class='biofold' >

        <span style="color:#d97b2d;font-size:150%">VarI</span>
        <span style="color:#d97b2d;font-size:50%">c</span><span style="font-size:50%">ommunity </span>
        <span style="color:#d97b2d;font-size:50%">o</span><span style="font-size:50%">f  </span>
        <span style="color:#d97b2d;font-size:50%">s</span><span style="font-size:50%">pecial </span>
        <span style="color:#d97b2d;font-size:50%">i</span><span style="font-size:50%">nterest </span>
        </div>
        <!--
        <div class='biofold' ><span style="color:#c31e27;">Bio</span><span style="color:#4689a3;">molecules</span> <span style="color:#c31e27;">Fol</span><span style="color:#4689a3;">ding and </span><span style="color:#c31e27;">D</span><span style="color:#4689a3;">isease</span></div><br>
        -->
	</td>
	<td width="20%">
	<div class="right_cell">
        <a href="./index.html">
        <img src="./img/abacov.jpg" width="150px" align="center">
        </a>
        </div>
	</td>
	</tr>
	</table>
  <br>
	<table width="100%">
        <tr>
        <td>
        <div class="form-inline justify-content-center">
        <nav>
          <ul class="nav nav-pills">
            <li role="presentation" ><a href="./index.html">Home</a></li>
            <li role="presentation" ><a href="./schedule.html">Schedule</a></li>
            <li role="presentation" class='active'><a href="">Submission</a></li>
            <li role="presentation" ><a href="./publication.html">Publication</a></li>
            <li role="presentation" ><a href="./company.html">Company</a></li>
	    <li role="presentation"><a href="./contact.html">Contact</a></li>
          </ul>
        </nav>
        </div>
        </td>
        </tr>
        </table>
    <br>
  	<br>

     <div class="text">
     <h4 class="title" >Submission</h4>
     </div>

  <div class="text">

    <table class="paper" border=0>
        <tbody><tr><td valign=top>
    <tr>
    <td valign=top>
    Presentation proposals and poster abstracts can be submitted through
    <a href="https://easychair.org/conferences/?conf=ismb2022abstracts#" target=_blank>EasyChair</a>.
    Submitted abstracts should describe original or recent scientific works in the areas listed below.
    <br><br>
    <ul>
    <li><span style="color:#d97b2d;font-weight:400;">Databases, data mining algorithms and visualization tools for variants analysis.</span><br>
      Current genomic databases contain millions of SNVs and vast amounts of related annotation data.
    Information continues to flow into these resources at ever increasing speeds. Making biologically relevant
    sense of this data deluge requires the development of better-defined data collection and access strategies.
    This topic is of particular interest to researchers working on storage, retrieval, and visualization of variation data.
    We will focus explicitly on two subsection of this topic: the development of SNV-related ontologies and
    methods for the automatic extraction of SNV data from literature.
    </ul><ul>
    <li><span style="color:#d97b2d;font-weight:400;">Methods for predicting regulatory/structural/functional impacts of SNVs.</span><br>
    Experimental study of functional effects of SNVs is complicated by a number of factors (e.g. compounding effects of variation
    in other genetic regions including linkage disequilibrium, problems with crystallization, expression related changes in
    phenotype, etc) and generally results in a very expensive and not very accurate estimation of real effect. A number of
    tools have been recently developed to evaluate regulatory, functional and structural effects of SNVs in silica.
    This topic encompasses the variants with protein and RNA effects, as well as variants in regulatory and functionally
    un-annotated regions. This year we would like to explicitly encourage submission relating to the studies of regulatory
    SNVs and genetic variation in drug response.
    </ul><ul>
    <li><span style="color:#d97b2d;font-weight:400;">Personal Genomics, GWAS studies and SNV prioritization.</span><br>
      The recent development in sequencing technologies has moved personal genomics (and thereby personalized medicine) much
      closer to reality. Genome wide association studies have become increasingly relied upon for disease-gene discoveries.
      Yet, a number of studies show that the discovered disorder associated SNVs do not account fully for the observed genetic
      risk. Thus, the GWAS should ideally provide preliminary genetic information, which could then be analyzed in silica in
      concert with other evidence (e.g. meta-analysis of a few GWAS or epistasis analysis within a single GWAS).
      In particular we will encourage the submissions addressing the detection of causative and marker SNVs and those tracking
      the possible relationships between different region of the genome.
    </ul><ul>
    <li><span style="color:#d97b2d;font-weight:400;">Population genomics and phylogenetic analysis.</span><br>
    Variation is the driving force of evolution. SNVs are the most common form of genetic variation.
    SNVs have been shown to be very useful for typing and resolving relationships between organisms of different species.
    This research topic would encompass the discussion of advantages and limitations of SNV based algorithms for the analysis
    of population genomic data and phylogenetically conserved genomic regions. We particularly encourage submissions discussing
    SNVs in animal and bacterial models, as well as metagenomic studies.


    </td></tr>
    </tbody>
    </table>
    </p>
    <br>
    <!---
    <div class="text">
    <h4 class="title" >Meeting program</h4>
    </div>
    <p class="text">
    TBA
    <p>
    <table border=0 width="100%">

    <tbody>
    <tr>
    <td  align=center>
    </td>
	  </tr>
	  <tbody>
	  </table>


    </p>

    <br>
    <br>
    -->
    <div class='footer-ok'><div>

      </div>
    </div> <!-- /container -->


    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <script src="../assets/js/ie10-viewport-bug-workaround.js"></script>
    -->
  </body>
</html>
