<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>VarI special interest group</title>
<style type="text/css">

body {
	background-color: #FE9901;
}

#dropmenudiv{
position:absolute;
border:1px WHITE;
border-bottom-width: 0;
font:normal 14px Verdana, Arial, Helvetica, sans-serif;
line-height:16px;
z-index:100;
}

#dropmenudiv a{
width: 220px;
display: block;
text-indent: 3px;
border-bottom: 1px WHITE;
padding: 1px 0;
text-decoration: none;
font-weight: bold;
}

#dropmenudiv a:hover{ /*hover background color*/
color: #79a631;
background-color: #ffffff;
}

</style>
<script type="text/javascript">
//Contents for menu 2, and so on
var menu2=new Array()
menu2[0]='<a href="topics.html">Topics</a>'
menu2[1]='<a href="kd.html">Key dates</a>'
menu2[2]='<a href="schedu.html">Schedule</a>'

var menu3=new Array()
menu3[0]='<a href="abstract.html">Talks and Posters</a>'
menu3[1]='<a href="company.html">Company Involvement</a>'
menu3[2]='<a href="special.html">Special Issue</a>'

var menu4=new Array()
menu4[0]='<a href="people.html">People</a>'
menu4[1]='<a href="conta.html">Contact</a>'

		
var menuwidth='165px' //default menu width
var menubgcolor='#FFFFFF'  //menu bgcolor
var disappeardelay=250  //menu disappear speed onMouseout (in miliseconds)
var hidemenu_onclick="yes" //hide menu when user clicks within menu?

/////No further editting needed

var ie4=document.all
var ns6=document.getElementById&&!document.all

if (ie4||ns6)
document.write('<div id="dropmenudiv" style="visibility:hidden;width:'+menuwidth+';background-color:'+menubgcolor+'" onMouseover="clearhidemenu()" onMouseout="dynamichide(event)"></div>')

function getposOffset(what, offsettype){
var totaloffset=(offsettype=="left")? what.offsetLeft : what.offsetTop;
var parentEl=what.offsetParent;
while (parentEl!=null){
totaloffset=(offsettype=="left")? totaloffset+parentEl.offsetLeft : totaloffset+parentEl.offsetTop;
parentEl=parentEl.offsetParent;
}
return totaloffset;
}

function showhide(obj, e, visible, hidden, menuwidth){
if (ie4||ns6)
dropmenuobj.style.left=dropmenuobj.style.top=-500
if (menuwidth!=""){
dropmenuobj.widthobj=dropmenuobj.style
dropmenuobj.widthobj.width=menuwidth
}
if (e.type=="click" && obj.visibility==hidden || e.type=="mouseover")
obj.visibility=visible
else if (e.type=="click")
obj.visibility=hidden
}

function iecompattest(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function clearbrowseredge(obj, whichedge){
var edgeoffset=0
if (whichedge=="rightedge"){
var windowedge=ie4 && !window.opera? iecompattest().scrollLeft+iecompattest().clientWidth-15 : window.pageXOffset+window.innerWidth-15
dropmenuobj.contentmeasure=dropmenuobj.offsetWidth
if (windowedge-dropmenuobj.x < dropmenuobj.contentmeasure)
edgeoffset=dropmenuobj.contentmeasure-obj.offsetWidth
}
else{
var topedge=ie4 && !window.opera? iecompattest().scrollTop : window.pageYOffset
var windowedge=ie4 && !window.opera? iecompattest().scrollTop+iecompattest().clientHeight-15 : window.pageYOffset+window.innerHeight-18
dropmenuobj.contentmeasure=dropmenuobj.offsetHeight
if (windowedge-dropmenuobj.y < dropmenuobj.contentmeasure){ //move up?
edgeoffset=dropmenuobj.contentmeasure+obj.offsetHeight
if ((dropmenuobj.y-topedge)<dropmenuobj.contentmeasure) //up no good either?
edgeoffset=dropmenuobj.y+obj.offsetHeight-topedge
}
}
return edgeoffset
}

function populatemenu(what){
if (ie4||ns6)
dropmenuobj.innerHTML=what.join("")
}

function dropdownmenu(obj, e, menucontents, menuwidth){
if (window.event) event.cancelBubble=true
else if (e.stopPropagation) e.stopPropagation()
clearhidemenu()
dropmenuobj=document.getElementById? document.getElementById("dropmenudiv") : dropmenudiv
populatemenu(menucontents)

if (ie4||ns6){
showhide(dropmenuobj.style, e, "visible", "hidden", menuwidth)
dropmenuobj.x=getposOffset(obj, "left")
dropmenuobj.y=getposOffset(obj, "top")
dropmenuobj.style.left=dropmenuobj.x-clearbrowseredge(obj, "rightedge")+"px"
dropmenuobj.style.top=dropmenuobj.y-clearbrowseredge(obj, "bottomedge")+obj.offsetHeight+"px"
}

return clickreturnvalue()
}

function clickreturnvalue(){
if (ie4||ns6) return false
else return true
}

function contains_ns6(a, b) {
while (b.parentNode)
if ((b = b.parentNode) == a)
return true;
return false;
}

function dynamichide(e){
if (ie4&&!dropmenuobj.contains(e.toElement))
delayhidemenu()
else if (ns6&&e.currentTarget!= e.relatedTarget&& !contains_ns6(e.currentTarget, e.relatedTarget))
delayhidemenu()
}

function hidemenu(e){
if (typeof dropmenuobj!="undefined"){
if (ie4||ns6)
dropmenuobj.style.visibility="hidden"
}
}

function delayhidemenu(){
if (ie4||ns6)
delayhide=setTimeout("hidemenu()",disappeardelay)
}

function clearhidemenu(){
if (typeof delayhide!="undefined")
clearTimeout(delayhide)
}

if (hidemenu_onclick=="yes")
document.onclick=hidemenu

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
</script>
<link rel="stylesheet" href="css/style.css" 
type="text/css" title="Snp sig">
</head>
<body onload="MM_preloadImages('img/bottoni-home2.gif')">
<div id="main">
<br />
<br />
<br />

<table width="880" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" class="style1">
  <tr>
    <td width="39" height="39" align="top"><img src="img/angolo.jpg" alt="" width="39" height="39" align="top" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-alt-ds.jpg" width="39" height="39" align="top" /></td>
  </tr>
  <tr>
    <td width="39">&nbsp;</td>
    <td><div id="content"><table border="0">

      <tr>
        <td colspan="3"><img src="img/vari.gif" width="524" height="110" /></td>
        <td><div align="right"><img src="img/abacov.jpg" width="162" height="110" /></div></td>
      </tr>
     <tr>
        <td ><br >
          </td>
        </tr>
      <tr>

        <td><div id="menu"><a href="index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image7','','img/bottoni-home2.gif',1)"><img src="img/bottoni-home.gif" name="Image7" width="45" height="22" border="0" id="Image7" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu2, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-prog.gif" width="78" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu3, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-sub.gif" width="113" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu4, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-organizz.gif" width="138" height="22" border="0" /></a></div></td>
      </tr>
      <tr>
        <td colspan="4"><p class="textzone"><strong><br><br>
              <img src="img/PU3.gif" width="17" height="14"><img src="img/PU1.gif" width="12" height="12"> VarI-COSI 2018 Meeting Programme
              <br>
              </strong>
              <br>
        </p>
          <table width="100%" border="0" cellpadding="3" cellspacing="3">
            <tr>
              <td><span class="textzone"><strong>10:15 - 10:20</strong></span></td>
              <td><span class="textzone"> Welcome from the committee </span></td>
            </tr>
            <tr>
              <td colspan="2"><span class="textzone"><br><br><strong>
                    <img src="img/PU.gif" width="12" height="12"> Session 1 
<!--
Annotation and prediction of structural/functional impacts of genetic variants.
--->
              </strong>
              <br><br></span></td>
            </tr>

              <td><span class="textzone"><strong>10:20 - 10:30</strong><br>&nbsp;
                </span></td>
              <td><span class="textzone">
                  <u>Elizabeth White.</u> University of Colorado, Denver (CO).
              <br><i>Integration of protein modifications, localizations, and pathways in a biological knowledge base.</i>
              </span></td>
            </tr>

            <tr>
              <td><span class="textzone"><strong>10:30 - 11:00</strong><br>&nbsp; </span></td>
              <td><span class="textzone"><b>Invited Speaker:</b>
              <a href="http://people.csail.mit.edu/bab/" target=blank>
              <strong>Bonnie Berger.</strong></a> Massachusetts Institute of Technology, Cambridge (MA).<br>
              <i>Genomic Crowdsourcing with Privacy.
	 	</i> 
              </span></td>
            </tr>
            <tr>

              <td><span class="textzone"><strong>11:00 - 11:20</strong><br>&nbsp;
		</span></td>
              <td><span class="textzone">
              <u>Yoonjoo Choi.</u> Korea Advanced Institute of Science and Technology, Daejeon, (South Korea).
              <br><i>DisruPPI: Structure-based computational redesign algorithm for protein 
		     binding disruption.</i>
              </span></td>
            </tr>
            
            <tr>
              <td><span class="textzone"><strong>11:20 - 11:40</strong><br>&nbsp;</span></td>
              <td><span class="textzone">

              <u>Maximilian Miller.</u> Rutgers University, New Brunswick (NJ).
              <br><i>
              Identifying protein positions for variation driven functional tuning.
              </i>

		</span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>11:40 - 12:00</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone">

              <u>Barthélémy Caron.</u> Institute Imagine, Paris (France).
	      <br><i>Prediction of non-coding variants causing Mendelian diseases through an integrative 
	      supervised<br>learning approach mining signals of ongoing purifying selection in humans.</i>
              </span></td>
            </tr>
         <tr>
              <td><span class="textzone"><strong>12:00 - 12:20</strong><br>&nbsp;</span></td>
              <td><span class="textzone">

		<u>Adriana Sperlea.</u> University of California, Los Angeles (CA).
			  <br><i>  
			Systematic Discovery of Conservation States for Single-Nucleotide Annotation of the Human Genome.
			  </i>

              </span></td>
            </tr>
         <tr>
              <td><span class="textzone"><strong>12:20 - 12:30</strong><br>&nbsp;</span></td>
              <td><span class="textzone">
		<u>Fabrizio Pucci.</u> Free University of Brussels (ULB), Brussels (Belgium).
              <br><i>Probing the effect of mutations on the interactome.</i>
              </span></td>
            </tr>
	    <tr>
              <td><span class="textzone"><strong>12:30 - 12:45</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone"><b>Company Presentation:</b> 

             <b><a href="https://www.qiagen.com/" target=blank>Jennifer Poitras - Qiagen.</a></b>
             <br>
              <i>Rapid Variant Interpretation and Reporting Using the QIAGEN Knowledgebase and QIAGEN Clinical <br>
	         Insight (QCI) Interpret.</i>

	      </span></td>
            </tr>
            
            <tr>
              <td><span class="textzone"><br /><strong>12:45 - 14:00</strong></span></td>
              <td><span class="textzone"><br /><b>Lunch Break and Poster Session with the Authors.</b>
              </span></td>
            </tr>
          
  
            <tr>
              <td colspan="2"><span class="textzone"><strong><br />
                  <br />
                    <img src="img/PU.gif" alt="" width="12" height="12" /> 
                    Session 2 
<!--
    Genetic variants as effectors of change: disease and evolution. <br />
-->
              <br />
              </strong></span></td>
              </tr>
            <tr>
              <td><span class="textzone"><strong>14:00 - 14:10</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone">
		<u>Farhad Hormozdiari.</u>  University of California, Los Angeles (CA).
              <br><i>Leveraging molecular quantitative trait loci to understand the genetic architecture of diseases 
              <br>and complex traits.</i>
              </span></td>
            </tr>

            <tr>
              <td><span class="textzone"><strong>14:10 - 14:40</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><strong>Invited Speaker:</strong> 
	      <a href="https://www.cs.princeton.edu/~mona/" target=blank><strong>Mona Singh.</strong></a>
              Princeton University, Princeton (NJ).
              <br>
               <i>
               Variation in protein interactions, with applications to disease.</i>
              </span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>14:40 - 15:00</strong><br>&nbsp;</span></td>
              <td><span class="textzone">
              <u>Lisa Gai.</u> University of California, Los Angeles (CA).
              <i><br> 
              Increasing the power of meta-analysis of genome-wide association studies to detect heterogeneous effects.
              </i>
			</span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>15:00 - 15:20</strong><br>&nbsp;</span></td>
              <td><span class="textzone">

              <u>Poulami Chaudhuri.</u> Innovation Lab, Tata Consultancy Services (India) <br>
              <i>An approach to discover branchpoint variants and assess the functional consequences.</i>
              <span></td>
            </tr>
	    <tr>
              <td><span class="textzone"><strong>15:20 - 15:40 </strong><br>&nbsp;&nbsp;</span></td>
              <td><span class="textzone">

              <u>Rachel Marty.</u> University of California, San Diego (CA). 
              <br><i>Computational modeling of genomic immune variation to deduce role of MHC Class 
                 II in cancer<br>susceptibility and evolution</i>

			  </span></td>
            </tr>
            
            <tr>
              <td><span class="textzone"><strong>15:40 - 15:50</strong><br>&nbsp;</span></td>
              <td><span class="textzone">
              <u>Emidio Capriotti.</u> University of Bologna, Bologna (Italy).
              <i><br> 
              Predicting gene expression changes in E. coli  from mRNA sequence information.
              </i>
			</span></td>
            </tr>
	     <tr>
              <td><span class="textzone"><strong>15:50 - 16:05</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><b>Company Presentation:</b>

             <b><a href="https://www.variantyx.com/" target=blank>Alexander Kaplun - Variantyx.</a></b>
             <br>
              <i>Detection of pathologic short tandem repeat extensions using Whole Genome Sequencing.</i>
              </span></td>
            </tr>
            <tr>
            <td><span class="textzone"><strong>16:05 - 16:40</strong></span></td>
            <td><span class="textzone">Coffee Break</span></td>
            </tr>

	   <tr>
              <td><span class="textzone"><strong>16:40 - 16:50 </strong><br>&nbsp;</span></td>
              <td><span class="textzone">
              <u>Cong Ma.</u> Carnegie Mellon University. Pittsburgh (PA).
              <br><i>
		       SQUID: Transcriptomic structural variation detection from RNA-seq.
              </i>
                          </span></td>
            </tr> 


	    <tr>
              <td><span class="textzone"><strong>16:50 - 17:20</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><strong>Invited Speaker:</strong>
	      <a href="https://www.cs.princeton.edu/people/profile/ogt" target=blank><strong>Olga Troyankaya.</strong></a> Princeton
              University, Princeton (NJ).
	      <br>
              <i>
               Decoding the human genome with deep learning models.</i>
              </span></td>
         <tr>
              <td><span class="textzone"><strong>17:20 - 17:40</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone">

		 <u>Hsuan-Lin Her.</u> Taipei Medical University, Taipei (Taiwan).
                          <br><i>
                         A pan-genome-based machine learning approach for predicting antimicrobial resistance activities
                         <br>of the Escherichia coli strains.

                          </i>

              </span></td>
            </tr>
         <tr>
              <td><span class="textzone"><strong>17:40 - 18:00</strong><br>&nbsp;<br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone">
		   <u>Francisco De La Vega.</u> Fabric Genomics Inc., Oakland (CA).
                          <br><i>
			Phenotype-driven variant prioritization significantly improves over impact and prevalence scores in 
			<br>a large-scale analysis of 2,408 cases of Mendelian disease diagnostics by whole-genome sequencing 
			<br> of ancestral genomes.
                          </i>
              </span></td>
            </tr>

<!--
            <tr>
              <td><span class="textzone"><strong>17:20 - 18:00</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><b>Round table discussion.</b><br>&nbsp; 
              
              </span></td>
            </tr>
-->
            <tr>
              <td><span class="textzone"><strong>18:00 - 18:10</strong></span></td>
              <td><span class="textzone">Closing remarks from the committee.</span></td>
            </tr>
			
          </table><br /><br /><br />
          </td>
      </tr>
	  
      <tr>
        <td colspan="4" class="footer" ></td>

      </tr>
    </table>
    </div>
    <!--End Content-->
      </td>
    <td width="39">&nbsp;</td>
  </tr>
  
  <tr>
    <td width="39" height="39"><img src="img/angolo-abasso-sin.jpg" width="39" height="39" /></td>

    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-abasso-ds.jpg" width="39" height="39" /></td>
  </tr>
</table>
<br>
<br>
</div>
</body>
</html>
