<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Vari special interest group</title>
<style type="text/css">

body {
	background-color: #FE9901;
}
#dropmenudiv{
position:absolute;
border:1px WHITE;
border-bottom-width: 0;
font:normal 14px Verdana, Arial, Helvetica, sans-serif;
line-height:16px;
z-index:100;
}

#dropmenudiv a{
width: 220px;
display: block;
text-indent: 3px;
border-bottom: 1px WHITE;
padding: 1px 0;
text-decoration: none;
font-weight: bold;
}

#dropmenudiv a:hover{ /*hover background color*/
color: #79a631;
background-color: #ffffff;
}
</style>
<script type="text/javascript">
//Contents for menu 2, and so on
var menu2=new Array()
menu2[0]='<a href="topics.html">Topics</a>'
menu2[1]='<a href="kd.html">Key dates</a>'
menu2[2]='<a href="schedu.html">Schedule</a>'

var menu3=new Array()
menu3[0]='<a href="abstract.html">Talks and Posters</a>'
menu3[1]='<a href="company.html">Company Involvement</a>'
menu3[2]='<a href="special.html">Special Issue</a>'

var menu4=new Array()
menu4[0]='<a href="people.html">People</a>'
menu4[1]='<a href="conta.html">Contact</a>'

		
var menuwidth='165px' //default menu width
var menubgcolor='#FFFFFF'  //menu bgcolor
var disappeardelay=250  //menu disappear speed onMouseout (in miliseconds)
var hidemenu_onclick="yes" //hide menu when user clicks within menu?

/////No further editting needed

var ie4=document.all
var ns6=document.getElementById&&!document.all

if (ie4||ns6)
document.write('<div id="dropmenudiv" style="visibility:hidden;width:'+menuwidth+';background-color:'+menubgcolor+'" onMouseover="clearhidemenu()" onMouseout="dynamichide(event)"></div>')

function getposOffset(what, offsettype){
var totaloffset=(offsettype=="left")? what.offsetLeft : what.offsetTop;
var parentEl=what.offsetParent;
while (parentEl!=null){
totaloffset=(offsettype=="left")? totaloffset+parentEl.offsetLeft : totaloffset+parentEl.offsetTop;
parentEl=parentEl.offsetParent;
}
return totaloffset;
}

function showhide(obj, e, visible, hidden, menuwidth){
if (ie4||ns6)
dropmenuobj.style.left=dropmenuobj.style.top=-500
if (menuwidth!=""){
dropmenuobj.widthobj=dropmenuobj.style
dropmenuobj.widthobj.width=menuwidth
}
if (e.type=="click" && obj.visibility==hidden || e.type=="mouseover")
obj.visibility=visible
else if (e.type=="click")
obj.visibility=hidden
}

function iecompattest(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function clearbrowseredge(obj, whichedge){
var edgeoffset=0
if (whichedge=="rightedge"){
var windowedge=ie4 && !window.opera? iecompattest().scrollLeft+iecompattest().clientWidth-15 : window.pageXOffset+window.innerWidth-15
dropmenuobj.contentmeasure=dropmenuobj.offsetWidth
if (windowedge-dropmenuobj.x < dropmenuobj.contentmeasure)
edgeoffset=dropmenuobj.contentmeasure-obj.offsetWidth
}
else{
var topedge=ie4 && !window.opera? iecompattest().scrollTop : window.pageYOffset
var windowedge=ie4 && !window.opera? iecompattest().scrollTop+iecompattest().clientHeight-15 : window.pageYOffset+window.innerHeight-18
dropmenuobj.contentmeasure=dropmenuobj.offsetHeight
if (windowedge-dropmenuobj.y < dropmenuobj.contentmeasure){ //move up?
edgeoffset=dropmenuobj.contentmeasure+obj.offsetHeight
if ((dropmenuobj.y-topedge)<dropmenuobj.contentmeasure) //up no good either?
edgeoffset=dropmenuobj.y+obj.offsetHeight-topedge
}
}
return edgeoffset
}

function populatemenu(what){
if (ie4||ns6)
dropmenuobj.innerHTML=what.join("")
}

function dropdownmenu(obj, e, menucontents, menuwidth){
if (window.event) event.cancelBubble=true
else if (e.stopPropagation) e.stopPropagation()
clearhidemenu()
dropmenuobj=document.getElementById? document.getElementById("dropmenudiv") : dropmenudiv
populatemenu(menucontents)

if (ie4||ns6){
showhide(dropmenuobj.style, e, "visible", "hidden", menuwidth)
dropmenuobj.x=getposOffset(obj, "left")
dropmenuobj.y=getposOffset(obj, "top")
dropmenuobj.style.left=dropmenuobj.x-clearbrowseredge(obj, "rightedge")+"px"
dropmenuobj.style.top=dropmenuobj.y-clearbrowseredge(obj, "bottomedge")+obj.offsetHeight+"px"
}

return clickreturnvalue()
}

function clickreturnvalue(){
if (ie4||ns6) return false
else return true
}

function contains_ns6(a, b) {
while (b.parentNode)
if ((b = b.parentNode) == a)
return true;
return false;
}

function dynamichide(e){
if (ie4&&!dropmenuobj.contains(e.toElement))
delayhidemenu()
else if (ns6&&e.currentTarget!= e.relatedTarget&& !contains_ns6(e.currentTarget, e.relatedTarget))
delayhidemenu()
}

function hidemenu(e){
if (typeof dropmenuobj!="undefined"){
if (ie4||ns6)
dropmenuobj.style.visibility="hidden"
}
}

function delayhidemenu(){
if (ie4||ns6)
delayhide=setTimeout("hidemenu()",disappeardelay)
}

function clearhidemenu(){
if (typeof delayhide!="undefined")
clearTimeout(delayhide)
}

if (hidemenu_onclick=="yes")
document.onclick=hidemenu

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
</script>
<link rel="stylesheet" href="css/style.css" 
type="text/css" title="Snp sig">
</head>
<body onload="MM_preloadImages('img/bottoni-home2.gif','img/submit1.gif')">
<div id="main">
<br />
<br />
<br />
<table width="880" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" class="style1">
  <tr>
    <td width="39" height="39" align="top"><img src="img/angolo.jpg" alt="" width="39" height="39" align="top" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-alt-ds.jpg" width="39" height="39" align="top" /></td>
  </tr>
  <tr>
    <td width="39">&nbsp;</td>
    <td><div id="content"><table border="0">
      <tr>
        <td colspan="3"><img src="img/vari.gif" width="524" height="110" /></td>
        <td><div align="right"><img src="img/abacov.jpg" width="162" height="110" /></div></td>
      </tr>
     <tr>
        <td ><br >
          </td>
        </tr>
      <tr>
        <td><div id="menu"><a href="index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image7','','img/bottoni-home2.gif',1)"><img src="img/bottoni-home.gif" name="Image7" width="45" height="22" border="0" id="Image7" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu2, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-prog.gif" width="78" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu3, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-sub.gif" width="113" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu4, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-organizz.gif" width="138" height="22" border="0" /></a></div></td>
      </tr>
      <tr>
        <td colspan="4"> <p class="textzone">
        <br />
        <strong>Submitted Presentation Proposals and Poster Abstracts describe original or recent scientific works in the areas listed below.<br />
          <br />
        </strong>
        <table width="100%" border="0">
          <tr>
            <td><strong>SUBMISSION OPTIONS:</strong> <br />
              <br />
             
              
                <li><a href='op.html'><strong>Presentation Proposal submissions</strong></a> <br />
                </li>
                <li><a href='pa.html'><strong>Poster Abstract submissions</strong></a></li>
              
              </td>
            <td valign="top"><div align="right"></div></td>
          </tr>
        </table>

	<br />
<!---
	<a href="https://www.easychair.org/conferences/?conf=varisig2016" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('submit5','','img/submit1.gif',1)" target="blank" class="mouse">
        <img src="img/submit.gif" alt="submit" name="submit5" width="85" height="30" border="0" id="submit5" /></a>

	<br /><br>
--->       
        <p class="textzone"><img src="img/PU.gif" width="12" height="12" />   <strong>Databases, data mining algorithms and visualization tools for genetic variant analysis</strong>.
        <br />
        Current genomic databases contain millions of SNVs and vast amounts of related
annotation data. Information continues to flow into these resources at ever
increasing speeds. Making biologically relevant sense of this data deluge requires
the development of better-defined data collection and access strategies. This topic is
of particular interest to researchers working on storage, retrieval, and visualization of
SNV related data. This year we will focus explicitly on two subsection of this topic:
the development of SNV-related ontologies and methods for the automatic extraction
of SNV data from literature.
        <br />
        <br />
        <br />
        <img src="img/PU1.gif" width="12" height="12" />  <strong>Methods for predicting regulatory/structural/functional impact of genetic variants</strong>.
        <br />
        Experimental study of functional effects of SNVs is complicated by a number of
factors (e.g. compounding effects of variation in other genetic regions including
linkage disequilibrium, problems with crystallization, expression related changes in
phenotype, etc) and generally results in a very expensive and not very accurate
estimation of real effect. A number of tools have been recently developed to
evaluate regulatory, functional and structural effects of SNVs in silico. This topic
encompasses the variants with protein and RNA effects, as well as variants in
regulatory and functionally un-annotated regions. This year we would like to explicitly
encourage submission relating to the studies of regulatory SNVs and genetic
variation in drug response.
        <br />
        <br />
        <br />
        <img src="img/PU.gif" alt="" width="12" height="12" />  <strong>Personal Genomics, GWAS studies and SNV prioritization</strong>.
        <br />
        The recent development in sequencing technologies has moved personal genomics
(and thereby personalized medicine) much closer to reality. Genome wide
association studies have become increasingly relied upon for disease-gene
discoveries. Yet, a number of studies show that the discovered disorder associated
SNVs do not account fully for the observed genetic risk. Thus, the GWAS should
ideally provide preliminary genetic information, which could then be analyzed in
silico in concert with other evidence (e.g. meta-analysis of a few GWAS or epistasis
analysis within a single GWAS). In particular we will encourage the submissions
addressing the detection of causative and marker SNVs and those tracking the
possible relationships between different region of the genome.
<br />
<br />
<br />
<img src="img/PU1.gif" alt="" width="12" height="12" /> <strong> Population genomics and phylogenetic analysis</strong>.
<br />
Variation is the driving force of evolution. SNVs are the most common form of
genetic variation. SNVs have been shown to be very useful for typing and resolving
relationships between organisms of different species. This research topic would
encompass the discussion of advantages and limitations of SNV based algorithms
for the analysis of population genomic data and phylogenetically conserved genomic
regions. We particularly encourage submissions discussing SNVs in animal and
bacterial models, as well as metagenomic studies.

        <br />
        <br />
        <br />
	<b>Note:</b> The presenter must be registered to attend the conference. 
        <br />
	<br />
            </p>
          </td>
      </tr>
      <tr>
        <td colspan="4" class="footer" ></td>
      </tr>
    </table>
    </div>
    <!--End Content-->
      </td>
    <td width="39">&nbsp;</td>
  </tr>
  
  <tr>
    <td width="39" height="39"><img src="img/angolo-abasso-sin.jpg" width="39" height="39" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-abasso-ds.jpg" width="39" height="39" /></td>
  </tr>
</table>
<br>
<br>
</div>
</body>
</html>
