<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>SNP special interest group</title>
<style type="text/css">

body {
	background-color: #FE9901;
}

#dropmenudiv{
position:absolute;
border:1px WHITE;
border-bottom-width: 0;
font:normal 14px Verdana, Arial, Helvetica, sans-serif;
line-height:16px;
z-index:100;
}

#dropmenudiv a{
width: 220px;
display: block;
text-indent: 3px;
border-bottom: 1px WHITE;
padding: 1px 0;
text-decoration: none;
font-weight: bold;
}

#dropmenudiv a:hover{ /*hover background color*/
color: #79a631;
background-color: #ffffff;
}

</style>
<script type="text/javascript">
//Contents for menu 2, and so on
var menu2=new Array()
menu2[0]='<a href="topics.html">Topics</a>'
menu2[1]='<a href="kd.html">Key dates</a>'
menu2[2]='<a href="schedu.html">Schedule</a>'

var menu3=new Array()
menu3[0]='<a href="abstract.html">Talks and Posters</a>'
menu3[1]='<a href="company.html">Company Involvement</a>'
menu3[2]='<a href="special.html">Special Issue</a>'

var menu4=new Array()
menu4[0]='<a href="people.html">People</a>'
menu4[1]='<a href="conta.html">Contact</a>'

		
var menuwidth='165px' //default menu width
var menubgcolor='#FFFFFF'  //menu bgcolor
var disappeardelay=250  //menu disappear speed onMouseout (in miliseconds)
var hidemenu_onclick="yes" //hide menu when user clicks within menu?

/////No further editting needed

var ie4=document.all
var ns6=document.getElementById&&!document.all

if (ie4||ns6)
document.write('<div id="dropmenudiv" style="visibility:hidden;width:'+menuwidth+';background-color:'+menubgcolor+'" onMouseover="clearhidemenu()" onMouseout="dynamichide(event)"></div>')

function getposOffset(what, offsettype){
var totaloffset=(offsettype=="left")? what.offsetLeft : what.offsetTop;
var parentEl=what.offsetParent;
while (parentEl!=null){
totaloffset=(offsettype=="left")? totaloffset+parentEl.offsetLeft : totaloffset+parentEl.offsetTop;
parentEl=parentEl.offsetParent;
}
return totaloffset;
}

function showhide(obj, e, visible, hidden, menuwidth){
if (ie4||ns6)
dropmenuobj.style.left=dropmenuobj.style.top=-500
if (menuwidth!=""){
dropmenuobj.widthobj=dropmenuobj.style
dropmenuobj.widthobj.width=menuwidth
}
if (e.type=="click" && obj.visibility==hidden || e.type=="mouseover")
obj.visibility=visible
else if (e.type=="click")
obj.visibility=hidden
}

function iecompattest(){
return (document.compatMode && document.compatMode!="BackCompat")? document.documentElement : document.body
}

function clearbrowseredge(obj, whichedge){
var edgeoffset=0
if (whichedge=="rightedge"){
var windowedge=ie4 && !window.opera? iecompattest().scrollLeft+iecompattest().clientWidth-15 : window.pageXOffset+window.innerWidth-15
dropmenuobj.contentmeasure=dropmenuobj.offsetWidth
if (windowedge-dropmenuobj.x < dropmenuobj.contentmeasure)
edgeoffset=dropmenuobj.contentmeasure-obj.offsetWidth
}
else{
var topedge=ie4 && !window.opera? iecompattest().scrollTop : window.pageYOffset
var windowedge=ie4 && !window.opera? iecompattest().scrollTop+iecompattest().clientHeight-15 : window.pageYOffset+window.innerHeight-18
dropmenuobj.contentmeasure=dropmenuobj.offsetHeight
if (windowedge-dropmenuobj.y < dropmenuobj.contentmeasure){ //move up?
edgeoffset=dropmenuobj.contentmeasure+obj.offsetHeight
if ((dropmenuobj.y-topedge)<dropmenuobj.contentmeasure) //up no good either?
edgeoffset=dropmenuobj.y+obj.offsetHeight-topedge
}
}
return edgeoffset
}

function populatemenu(what){
if (ie4||ns6)
dropmenuobj.innerHTML=what.join("")
}

function dropdownmenu(obj, e, menucontents, menuwidth){
if (window.event) event.cancelBubble=true
else if (e.stopPropagation) e.stopPropagation()
clearhidemenu()
dropmenuobj=document.getElementById? document.getElementById("dropmenudiv") : dropmenudiv
populatemenu(menucontents)

if (ie4||ns6){
showhide(dropmenuobj.style, e, "visible", "hidden", menuwidth)
dropmenuobj.x=getposOffset(obj, "left")
dropmenuobj.y=getposOffset(obj, "top")
dropmenuobj.style.left=dropmenuobj.x-clearbrowseredge(obj, "rightedge")+"px"
dropmenuobj.style.top=dropmenuobj.y-clearbrowseredge(obj, "bottomedge")+obj.offsetHeight+"px"
}

return clickreturnvalue()
}

function clickreturnvalue(){
if (ie4||ns6) return false
else return true
}

function contains_ns6(a, b) {
while (b.parentNode)
if ((b = b.parentNode) == a)
return true;
return false;
}

function dynamichide(e){
if (ie4&&!dropmenuobj.contains(e.toElement))
delayhidemenu()
else if (ns6&&e.currentTarget!= e.relatedTarget&& !contains_ns6(e.currentTarget, e.relatedTarget))
delayhidemenu()
}

function hidemenu(e){
if (typeof dropmenuobj!="undefined"){
if (ie4||ns6)
dropmenuobj.style.visibility="hidden"
}
}

function delayhidemenu(){
if (ie4||ns6)
delayhide=setTimeout("hidemenu()",disappeardelay)
}

function clearhidemenu(){
if (typeof delayhide!="undefined")
clearTimeout(delayhide)
}

if (hidemenu_onclick=="yes")
document.onclick=hidemenu

function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
function MM_swapImgRestore() { //v3.0
  var i,x,a=document.MM_sr; for(i=0;a&&i<a.length&&(x=a[i])&&x.oSrc;i++) x.src=x.oSrc;
}
function MM_findObj(n, d) { //v4.01
  var p,i,x;  if(!d) d=document; if((p=n.indexOf("?"))>0&&parent.frames.length) {
    d=parent.frames[n.substring(p+1)].document; n=n.substring(0,p);}
  if(!(x=d[n])&&d.all) x=d.all[n]; for (i=0;!x&&i<d.forms.length;i++) x=d.forms[i][n];
  for(i=0;!x&&d.layers&&i<d.layers.length;i++) x=MM_findObj(n,d.layers[i].document);
  if(!x && d.getElementById) x=d.getElementById(n); return x;
}

function MM_swapImage() { //v3.0
  var i,j=0,x,a=MM_swapImage.arguments; document.MM_sr=new Array; for(i=0;i<(a.length-2);i+=3)
   if ((x=MM_findObj(a[i]))!=null){document.MM_sr[j++]=x; if(!x.oSrc) x.oSrc=x.src; x.src=a[i+2];}
}
</script>
<link rel="stylesheet" href="css/style.css" 
type="text/css" title="Snp sig">
</head>
<body onload="MM_preloadImages('img/bottoni-home2.gif')">
<div id="main">
<br />
<br />
<br />

<table width="880" border="0" align="center" cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" class="style1">
  <tr>
    <td width="39" height="39" align="top"><img src="img/angolo.jpg" alt="" width="39" height="39" align="top" /></td>
    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-alt-ds.jpg" width="39" height="39" align="top" /></td>
  </tr>
  <tr>
    <td width="39">&nbsp;</td>
    <td><div id="content"><table border="0">

      <tr>
        <td colspan="3"><img src="img/snp.gif" width="524" height="110" /></td>
        <td><div align="right"><img src="img/abaco5.jpg" width="162" height="110" /></div></td>
      </tr>
     <tr>
        <td ><br >
          </td>
        </tr>
      <tr>

        <td><div id="menu"><a href="index.html" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image7','','img/bottoni-home2.gif',1)"><img src="img/bottoni-home.gif" name="Image7" width="45" height="22" border="0" id="Image7" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu2, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-prog.gif" width="78" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu3, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-sub.gif" width="113" height="22" border="0" /></a></div></td>
        <td><div id="menu"><a href="#" onClick="return clickreturnvalue()" onMouseover="dropdownmenu(this, event, menu4, '165px')" onMouseout="delayhidemenu()"><img src="img/bottoni-organizz.gif" width="138" height="22" border="0" /></a></div></td>
      </tr>
      <tr>
        <td colspan="4"><p class="textzone"><strong><br><br>
              <img src="img/PU3.gif" width="17" height="14" /><img src="img/PU1.gif" width="12" height="12" /> SNP-SIG 2012 Meeting Programme  
	</i> 	      <br />
              <br />
        </strong>
        </p>
          <table width="100%" border="0" cellpadding="3" cellspacing="3">
            <tr>
              <td><span class="textzone"><strong>08:20 - 08:30</strong></span></td>
              <td><span class="textzone"> Welcome from the committee </span></td>
            </tr>
            <tr>
              <td colspan="2"><span class="textzone"><br /><br /><strong>
                    <img src="img/PU.gif" width="12" height="12" /> Session 1: Annotation and prediction of structural/functional impacts of coding SNPs
              </strong>
              <br /><br /></span></td>
              </tr>

            <tr>
              <td><span class="textzone"><strong>08:30 - 09:20</strong><br>&nbsp; </span></td>
              <td><span class="textzone"><b>Highlight Speaker:</b> 
              <a href="http://www.gene.com/gene/gred/people/zeminzhang.php" target=blank>
              <strong>Zemin Zhang</strong></a>, Genentech, San Francisco (USA)<br>
              <i>Identification and computational analysis of somatic 
              mutations in cancer samples.</i> 
              </span></td>
            </tr>
            <tr>

              <td><span class="textzone"><strong>09:20 - 09:45</strong><br>&nbsp;
		</span></td>
              <td><span class="textzone">
              Gustavo Parisi. Universidad Nacional de Quilmes, Buenos Aires (Argentina)
              <br><i>Improving the prediction of disease-related variants using protein dynamism.</i>
              </span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>09:45 - 10:10</strong><br>&nbsp;</span></td>
              <td><span class="textzone">
              Maricel Kann. University of Maryland, Baltimore (USA)
              <br><i>
			  Protein Domain-Centric Approach to Study of Cancer Somatic Mutations.
			  </i>
			  </span></td>

            </tr>
            <tr>
              <td><span class="textzone"><strong>10:10 - 10:30</strong></span></td>
              <td><span class="textzone">Coffee Break</span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>10:30 - 10:55</strong><br>&nbsp;</span></td>

              <td><span class="textzone">
              Lei Xie. Hunter College, New York (USA)
			  <br><i>Multiscale Modeling of Causal Phenotypic Associations of nsSNPs: Application<br>
			   to Hypoxia.</i>
              </span></td>
            </tr>
         <tr>
              <td><span class="textzone"><strong>10:55 - 11:20</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone">
              Nouf Alnumair. University College London (UK)
			  <br><i>The SAAP database and other tools to analyze the impact of mutations and predict 
			  <br>the pathogenicity of a novel mutation.
			  </i>
              </span></td>
            </tr>
          <tr>
              <td><span class="textzone"><strong>11:20 - 11:45</strong><br>&nbsp;</span></td>
              <td><span class="textzone">
              Frank Schacherer. BIOBASE GmbH.
              <br><i>
              Predicting the disease potential of gene mutations with MutationTaster
			  </i>
              </span></td>
            </tr>
         <tr>

              <td><span class="textzone"><strong>11:45 - 12:25</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><b>Keynote Presentation:</b> 
              <a href="http://www.bcm.edu/genetics/index.cfm?pmid=10943" target=blank><strong>
              Oliver Lichtarge</strong></a>, Baylor College of Medicine, Houston (USA).<br>
              <i>Missense Mutations in Action: an Analytic View</i>
             </span></td>
            </tr>
            
	    <tr>
              <td><span class="textzone"><strong>12:25 - 12:40</strong></span></td>
              <td><span class="textzone"><b>Company Presentation</b> 
              <b><a href="http://www.biobase-international.com/index.php?id=322" target=blank>
              Frank Schacherer</a></b>, BIOBASE GmbH.
	      </span></td>
            </tr>
            
            <tr>
              <td><span class="textzone"><br /><strong>12:40 - 13:20</strong></span></td>
              <td><span class="textzone"><br /><b>Lunch Break and Poster Session with the Authors</b>
              </span></td>

            </tr>
            <tr>
              <td colspan="2"><span class="textzone"><strong><br />
                  <br />
                    <img src="img/PU.gif" alt="" width="12" height="12" /> 
                    Session II: SNPs as effectors of change: disease and evolution <br />
              <br />
              </strong></span></td>
              </tr>

            <tr>
              <td><span class="textzone"><strong>13:20 - 14:10</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><strong>Highlight Presentation:</strong> 
              <a href="http://www.cbse.ucsc.edu/people/haussler" target=blank><strong>
              David Haussler</strong></a>, University of California Santa Cruz (USA).<br>
               <i>Somatic mutations in cancer as assessed by whole genome sequencing</i>
              </span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>14:10 - 14:35</strong><br>&nbsp;<br>&nbsp;</span></td>

              <td><span class="textzone">
              Boris Reva. Memorial Sloan-Kettering Cancer Center, New York (USA)
              <br><i>
              Revealing selection in cancer using the predicted functional impact of cancer mutations.
              <br>Application to nomination of cancer drivers 
			  </i>
              </span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>14:35 - 15:00</strong><br>&nbsp;</span></td>
              <td><span class="textzone">
              Janita Thusberg. Buck Institute, Novato (USA).
              <i><br>
			  Predicting Pharmacogenetic Protein Variants.
			  </i>
			</span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>15:00 - 15:25</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span class="textzone">
              Adam Kowalczyk. University of Melbourne (Australia).
			  <br><i>Detection and analysis of interactions in multiple GWAS studies using standard 
			  <br> desktop PC.</i>
<span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>15:25 - 15:45</strong></span></td>
              <td><span class="textzone">Coffee Break</span></td>
            </tr>
            <tr>
              <td><span class="textzone"><strong>15:45 - 16:10 </strong></span></td>
              <td><span class="textzone">
              Yana Bromberg. Rutgers University, New Brunswick (USA)
              <br><i>Noise in Biology: defining the neutral range of protein function.
              </i>
			  </span></td>

            </tr>
            <tr>
              <td><span class="textzone"><strong>16:10 - 16:50</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><b>Keynote Presentation:</b></span> 
              <a href="http://www-helix.stanford.edu/people/altman/" target=blank><strong>
              Russ Altman</strong></a>, Stanford University (USA).<br>
              <i>Defining levels of evidence supporting clinically actionable SNP associations.</i> 
              </td>
            </tr>

			<tr>
              <td><span class="textzone"><strong>16:50 - 17:20</strong>&nbsp;</span></td>
              <td><span class="textzone"><b>CAGI corner: </b> 
              <b><a href="http://compbio.berkeley.edu/people/brenner/" target=blank>Steven Brenner</a></b>, 
              University of California Berkeley (USA) 
              </span></td>
            </tr>

            <tr>
              <td><span class="textzone"><strong>17:20 - 18:10</strong><br>&nbsp;</span></td>
              <td><span class="textzone"><b>Round Table discussion:</b><br> 
              <b><a href="http://www.ikmb.uni-kiel.de/cms/en/about-us/staff/staffpage/andre-franke/" target=blank>Andre Franke<a/></b>, 
              <b><a href="http://karchinlab.org/" target=blank>Rachel Karchin</a></b>, 
              <b><a href="http://www.buckinstitute.org/mooneylab" target=blank>Sean Mooney</a></b>,  
              <b><a href="http://genetics.bwh.harvard.edu/genetics/labs/Sunyaev/index.html" target=blank>
              Shamil Sunyaev</a></b>.
              </span></td>
            </tr>

            <tr>
              <td><span class="textzone"><strong>18:10 - 18:20</strong></span></td>
              <td><span class="textzone">Closing remarks from the committee</span></td>
            </tr>
			
          </table><br /><br /><br />
          </td>
      </tr>
	  
      <tr>
        <td colspan="4" class="footer" >SNP-SIG ISMB 2012, Long Beach (CA), USA.</td>

      </tr>
    </table>
    </div>
    <!--End Content-->
      </td>
    <td width="39">&nbsp;</td>
  </tr>
  
  <tr>
    <td width="39" height="39"><img src="img/angolo-abasso-sin.jpg" width="39" height="39" /></td>

    <td>&nbsp;</td>
    <td width="39" height="39"><img src="img/angolo-abasso-ds.jpg" width="39" height="39" /></td>
  </tr>
</table>
</div>
</body>
</html>
