<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Variant Interpretation COSI">
    <meta name="author" content="Emidio Capriotti">
    <link rel="shortcut icon" href="../favicon.ico" type="image/vnd.microsoft.icon" />

    <title>Variant Interpretation - COSI </title>

    <!-- Bootstrap core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <link href="../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    -->

    <!-- Custom styles for this template -->
    <link href="../css/jumbotron-narrow-biofold.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
    <script src="../assets/js/ie-emulation-modes-warning.js"></script>
    -->



  </head>

  <body>
    <br>
    <div class="container">

      <div class="jumbotron">
        <table width="100%">
        <tr>
        <td width="80%">
	<div class='biofold' >

        <span style="color:#d97b2d;font-size:150%">VarI</span>
        <span style="color:#d97b2d;font-size:50%">c</span><span style="font-size:50%">ommunity </span>
        <span style="color:#d97b2d;font-size:50%">o</span><span style="font-size:50%">f  </span>
        <span style="color:#d97b2d;font-size:50%">s</span><span style="font-size:50%">pecial </span>
        <span style="color:#d97b2d;font-size:50%">i</span><span style="font-size:50%">nterest </span>

        </div>
        <!--
        <div class='biofold' ><span style="color:#c31e27;">Bio</span><span style="color:#4689a3;">molecules</span> <span style="color:#c31e27;">Fol</span><span style="color:#4689a3;">ding and </span><span style="color:#c31e27;">D</span><span style="color:#4689a3;">isease</span></div><br>
        -->
	</td>
	<td width="20%">
	<div class="right_cell">
        <a href="../index.html">
        <img src="../img/abacov.jpg" width="150px" align="center">
        </a>
      </div>
	</td>
	</tr>
	</table>
  <br>
	<table width="100%">
        <tr>
        <td>
        <div class="form-inline justify-content-center">
        <nav>
          <ul class="nav nav-pills">
            <li role="presentation"><a href="../index.html">Home</a></li>
            <li role="presentation" class='active'><a href="../schedule.html">Schedule</a></li>
            <li role="presentation"><a href="../submission.html">Submission</a></li>
            <li role="presentation"><a href="../publication.html">Publication</a></li>
            <li role="presentation"><a href="../company.html">Company</a></li>
	    <li role="presentation"><a href="../contact.html">Contact</a></li>
          </ul>
        </nav>
        </div>
        </td>
        </tr>
        </table>
    <br>
  	<br>

     <div class="text">
     <h4 class="title" >VarI COSI 2018</h4>
     </div>
     <br>
     <p class="text">
       <span style="color:#d97b2d;font-weight:400;">What:</span> Variant Interpretation Session at the ISMB Conference 2018<br>
       <span style="color:#d97b2d;font-weight:400;">When:</span> 10:15 AM - 6:00 PM, July 8th, 2018<br>
       <span style="color:#d97b2d;font-weight:400;">Where:</span> Room Columbus KL, Hyatt Regency Hotel, Chicago (IL), USA.

     </p>
      <table  border="0"  cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" >
        <colgroup>
          <col style="width:15%">
          <col style="width:85%">
          </colgroup>
        <tbody>
          <tr>
            <td colspan="2"><span><strong><br />
                <br /><p class="text-small">
                  Session 1
            <br /><br />
            </strong></span></td>
            </tr>

            <tr>
              <td><span ><p class="text-small"><strong>10:15 - 10:20</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small"> Welcome from the committee </span><br>&nbsp;</td>
            </tr>

            <td><span ><p class="text-small"><strong>10:20 - 10:30</strong><br>&nbsp;
              </span></td>
            <td><span ><p class="text-small">
                <u>Elizabeth White.</u> University of Colorado, Denver (CO).
            <br><i>Integration of protein modifications, localizations, and pathways in a biological knowledge base.</i>
            </span></td>
          </tr>

          <tr>
            <td><span ><p class="text-small"><strong>10:30 - 11:00</strong><br>&nbsp; </span></td>
            <td><span ><p class="text-small"><b>Invited Speaker:</b>
            <a href="http://people.csail.mit.edu/bab/" target=blank>
            <strong>Bonnie Berger.</strong></a> Massachusetts Institute of Technology, Cambridge (MA).<br>
            <i>Genomic Crowdsourcing with Privacy.
  </i>
            </span></td>
          </tr>
          <tr>

            <td><span ><p class="text-small"><strong>11:00 - 11:20</strong><br>&nbsp;
  </span></td>
            <td><span ><p class="text-small">
            <u>Yoonjoo Choi.</u> Korea Advanced Institute of Science and Technology, Daejeon, (South Korea).
            <br><i>DisruPPI: Structure-based computational redesign algorithm for protein
       binding disruption.</i>
            </span></td>
          </tr>

          <tr>
            <td><span ><p class="text-small"><strong>11:20 - 11:40</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">

            <u>Maximilian Miller.</u> Rutgers University, New Brunswick (NJ).
            <br><i>
            Identifying protein positions for variation driven functional tuning.
            </i>

  </span></td>
          </tr>
          <tr>
            <td><span ><p class="text-small"><strong>11:40 - 12:00</strong><br>&nbsp;<br>&nbsp;</span></td>
            <td><span ><p class="text-small">

            <u>Barthélémy Caron.</u> Institute Imagine, Paris (France).
      <br><i>Prediction of non-coding variants causing Mendelian diseases through an integrative
      supervised<br>learning approach mining signals of ongoing purifying selection in humans.</i>
            </span></td>
          </tr>
       <tr>
            <td><span ><p class="text-small"><strong>12:00 - 12:20</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">

  <u>Adriana Sperlea.</u> University of California, Los Angeles (CA).
      <br><i>
    Systematic Discovery of Conservation States for Single-Nucleotide Annotation of the Human Genome.
      </i>

            </span></td>
          </tr>
       <tr>
            <td><span ><p class="text-small"><strong>12:20 - 12:30</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">
  <u>Fabrizio Pucci.</u> Free University of Brussels (ULB), Brussels (Belgium).
            <br><i>Probing the effect of mutations on the interactome.</i>
            </span></td>
          </tr>
    <tr>
            <td><span ><p class="text-small"><strong>12:30 - 12:45</strong><br>&nbsp;<br>&nbsp;</span></td>
            <td><span ><p class="text-small"><b>Company Presentation:</b>

           <b><a href="https://www.qiagen.com/" target=blank>Jennifer Poitras - Qiagen.</a></b>
           <br>
            <i>Rapid Variant Interpretation and Reporting Using the QIAGEN Knowledgebase and QIAGEN Clinical <br>
         Insight (QCI) Interpret.</i>

      </span></td>
          </tr>

          <tr>
            <td><span ><p class="text-small"><br /><strong>12:45 - 14:00</strong></span></td>
            <td><span ><p class="text-small"><br /><b>Lunch Break and Poster Session with the Authors.</b>
            </span></td>
          </tr>


          <tr>
            <td colspan="2"><span ><strong><br />
                <br />
                  <p class="text-small">
                  Session 2<br>&nbsp;
<!--
  Genetic variants as effectors of change: disease and evolution. <br />
-->
            <br />
            </strong></span></td>
            </tr>
          <tr>
            <td><span ><p class="text-small"><strong>14:00 - 14:10</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">
  <u>Farhad Hormozdiari.</u>  University of California, Los Angeles (CA).
            <br><i>Leveraging molecular quantitative trait loci to understand the genetic architecture of diseases
            and complex traits.</i>
            </span></td>
          </tr>

          <tr>
            <td><span ><p class="text-small"><strong>14:10 - 14:40</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small"><strong>Invited Speaker:</strong>
      <a href="https://www.cs.princeton.edu/~mona/" target=blank><strong>Mona Singh.</strong></a>
            Princeton University, Princeton (NJ).
            <br>
             <i>
             Variation in protein interactions, with applications to disease.</i>
            </span></td>
          </tr>
          <tr>
            <td><span ><p class="text-small"><strong>14:40 - 15:00</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">
            <u>Lisa Gai.</u> University of California, Los Angeles (CA).
            <i><br>
            Increasing the power of meta-analysis of genome-wide association studies to detect heterogeneous effects.
            </i>
    </span></td>
          </tr>
          <tr>
            <td><span ><p class="text-small"><strong>15:00 - 15:20</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">

            <u>Poulami Chaudhuri.</u> Innovation Lab, Tata Consultancy Services (India) <br>
            <i>An approach to discover branchpoint variants and assess the functional consequences.</i>
            <span></td>
          </tr>
    <tr>
            <td><span ><p class="text-small"><strong>15:20 - 15:40 </strong><br>&nbsp;&nbsp;</span></td>
            <td><span ><p class="text-small">

            <u>Rachel Marty.</u> University of California, San Diego (CA).
            <br><i>Computational modeling of genomic immune variation to deduce role of MHC Class
               II in cancer<br>susceptibility and evolution</i>

      </span></td>
          </tr>

          <tr>
            <td><span ><p class="text-small"><strong>15:40 - 15:50</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">
            <u>Emidio Capriotti.</u> University of Bologna, Bologna (Italy).
            <i><br>
            Predicting gene expression changes in E. coli  from mRNA sequence information.
            </i>
    </span></td>
          </tr>
     <tr>
            <td><span ><p class="text-small"><strong>15:50 - 16:05</strong><br>&nbsp;<br>&nbsp;</span></td>
            <td><span ><p class="text-small"><b>Company Presentation:</b>

           <b><a href="https://www.variantyx.com/" target=blank>Alexander Kaplun - Variantyx.</a></b>
           <br>
            <i>Detection of pathologic short tandem repeat extensions using Whole Genome Sequencing.</i>
            </span><br>&nbsp;</td>
          </tr>
          <tr>
          <td><span ><p class="text-small"><strong>16:05 - 16:40</strong></span><br>&nbsp;</td>
          <td><span ><p class="text-small"><strong>Coffee Break</strong></span><br>&nbsp;</td>
          </tr>

   <tr>
            <td><span ><p class="text-small"><strong>16:40 - 16:50 </strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">
            <u>Cong Ma.</u> Carnegie Mellon University. Pittsburgh (PA).
            <br><i>
         SQUID: Transcriptomic structural variation detection from RNA-seq.
            </i>
                        </span></td>
          </tr>


    <tr>
            <td><span ><p class="text-small"><strong>16:50 - 17:20</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small"><strong>Invited Speaker:</strong>
      <a href="https://www.cs.princeton.edu/people/profile/ogt" target=blank><strong>Olga Troyankaya.</strong></a> Princeton
            University, Princeton (NJ).
      <br>
            <i>
             Decoding the human genome with deep learning models.</i>
            </span></td>
       <tr>
            <td><span ><p class="text-small"><strong>17:20 - 17:40</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small">

   <u>Hsuan-Lin Her.</u> Taipei Medical University, Taipei (Taiwan).
                        <br><i>
                       A pan-genome-based machine learning approach for predicting antimicrobial resistance activities
                       of the Escherichia coli strains.

                        </i>

            </span></td>
          </tr>
       <tr>
            <td><span ><p class="text-small"><strong>17:40 - 18:00</strong><br>&nbsp;<br>&nbsp;</span></td>
            <td><span ><p class="text-small">
     <u>Francisco De La Vega.</u> Fabric Genomics Inc., Oakland (CA).
                        <i>
    Phenotype-driven variant prioritization significantly improves over impact and prevalence scores in
    a large-scale analysis of 2,408 cases of Mendelian disease diagnostics by whole-genome sequencing
    of ancestral genomes.<br>&nbsp;
                        </i>
            </span></td>
          </tr>


          <tr>
            <td><span ><p class="text-small"><strong>18:00 - 18:10</strong></span></td>
            <td><span ><p class="text-small">Closing remarks from the committee.</span></td>
          </tr>
  						<!--
              <tr>
                <td><span ><strong>18:05 - 20:00</strong></span></td>
                <td><span ><b>Online Poster session with the authors.</b></span></td>
              </tr>
  					-->
          </tbody>
        </table>


    <br>
    <br>

    <div class='footer-ok'><div>

      </div>
    </div> <!-- /container -->


    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <script src="../assets/js/ie10-viewport-bug-workaround.js"></script>
    -->
  </body>
</html>
