<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Variant Interpretation COSI">
    <meta name="author" content="Emidio Capriotti">
    <link rel="shortcut icon" href="../favicon.ico" type="image/vnd.microsoft.icon" />

    <title>Variant Interpretation - COSI </title>

    <!-- Bootstrap core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <link href="../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    -->

    <!-- Custom styles for this template -->
    <link href="../css/jumbotron-narrow-biofold.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
    <script src="../assets/js/ie-emulation-modes-warning.js"></script>
    -->



  </head>

  <body>
    <br>
    <div class="container">

      <div class="jumbotron">
        <table width="100%">
        <tr>
        <td width="80%">
	<div class='biofold' >

        <span style="color:#d97b2d;font-size:150%">VarI</span>
        <span style="color:#d97b2d;font-size:50%">c</span><span style="font-size:50%">ommunity </span>
        <span style="color:#d97b2d;font-size:50%">o</span><span style="font-size:50%">f  </span>
        <span style="color:#d97b2d;font-size:50%">s</span><span style="font-size:50%">pecial </span>
        <span style="color:#d97b2d;font-size:50%">i</span><span style="font-size:50%">nterest </span>

        </div>
        <!--
        <div class='biofold' ><span style="color:#c31e27;">Bio</span><span style="color:#4689a3;">molecules</span> <span style="color:#c31e27;">Fol</span><span style="color:#4689a3;">ding and </span><span style="color:#c31e27;">D</span><span style="color:#4689a3;">isease</span></div><br>
        -->
	</td>
	<td width="20%">
	<div class="right_cell">
        <a href="../index.html">
        <img src="../img/abacov.jpg" width="150px" align="center">
        </a>
      </div>
	</td>
	</tr>
	</table>
  <br>
	<table width="100%">
        <tr>
        <td>
        <div class="form-inline justify-content-center">
        <nav>
          <ul class="nav nav-pills">
            <li role="presentation"><a href="../index.html">Home</a></li>
            <li role="presentation" class='active'><a href="../schedule.html">Schedule</a></li>
            <li role="presentation"><a href="../submission.html">Submission</a></li>
            <li role="presentation"><a href="../publication.html">Publication</a></li>
            <li role="presentation"><a href="../company.html">Company</a></li>
	    <li role="presentation"><a href="../contact.html">Contact</a></li>
          </ul>
        </nav>
        </div>
        </td>
        </tr>
        </table>
    <br>
  	<br>

     <div class="text">
     <h4 class="title" >VarI COSI 2017</h4>
     </div>
     <br>
     <p class="text">
       <span style="color:#d97b2d;font-weight:400;">What:</span> Variant Interpretation Session at the ISMB/ECCB Conference 2017<br>
       <span style="color:#d97b2d;font-weight:400;">When:</span> 10:00 AM - 6:00 PM, July 24th, 2017<br>
       <span style="color:#d97b2d;font-weight:400;">Where:</span> Hall IA, Prague Convention Center, Prague, Czech Republic.
     </p>
      <table  border="0"  cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" >
        <colgroup>
          <col style="width:15%">
          <col style="width:85%">
          </colgroup>
        <tbody>
          <tr>
            <td colspan="2"><span><strong><br />
                <br /><p class="text-small">
                  Session 1
            <br /><br />
            </strong></span></td>
            </tr>

              <tr>
                <td><span ><p class="text-small"><strong>10:00 - 10:10</strong><br>&nbsp;</span></td>
                <td><span ><p class="text-small"> Welcome from the committee<br>&nbsp;</span></td>
              </tr>

              <tr>
              <td><span  ><p class="text-small"><strong>10:10 - 10:40</strong><br>&nbsp; </span></td>
              <td><span  ><p class="text-small"><b>Highlight Speaker:</b>
              <a href="http://www.professoren.tum.de/en/rost-burkhard/" target=blank>
              <strong>Burkhard Rost.</strong></a> Techincal University of Munich, Munich (Germany).<br>
              <i>Common sequence variants affect molecular function more than
		rare variants?</i>
              </span></td>
            </tr>
            <tr>

              <td><span ><p class="text-small"><strong>10:40 - 11:00</strong><br>&nbsp;
		</span></td>
              <td><span ><p class="text-small">
              <u>Maximilian Miller.</u> Rutgers University, New Brunswick (NJ).
              <br><i>Computational predictors fail to identify amino acid substitution
		     effects at rheostat positions.</i>
              </span></td>
            </tr>

            <tr>
              <td><span ><p class="text-small"><strong>11:00 - 11:20</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small">
              <u>Kymberleigh Pagel.</u> Indiana University, Bloomington (IN).
              <br><i>
              When loss-of-function is loss of function: assessing mutational
              signatures and impact of loss-of-function genetic variants.

              </i>
		</span></td>
            </tr>
            <tr>
              <td><span ><p class="text-small"><strong>11:20 - 11:40</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small">
              <u>Martin Kircher.</u> Berlin Institute of Health, Berlin (Germany).
	      <br><i>Assessing regulatory variant effect scores
                     by massively parallel reporter assays.</i>
              </span></td>
            </tr>
         <tr>
              <td><span ><p class="text-small"><strong>11:40 - 12:00</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small"><u>Emidio Capriotti.</u> University of Bologna, Bologna (Italy).
			  <br><i>
			PhD-SNPg: A webserver and lightweight tool for scoring single nucleotide variants.
			  </i>
              </span></td>
            </tr>
         <tr>
              <td><span ><p class="text-small"><strong>12:00 - 12:20</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small"><u>Imane Boudellioua.</u> King Abdullah University of Science and Technology, Thuwal (Saudi Arabia).
              <br><i>Phenotype-driven discovery of digenic variants in personal
                     genome sequences.</i>
              </span></td>
            </tr>
	    <tr>
              <td><span ><p class="text-small"><strong>12:20 - 12:35</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span ><p class="text-small"><b>Company Presentation:</b>
             <b><a href="https://www.qiagen.com/" target=blank>Anika Joeker - Qiagen.</a></b>
             <br>
              <i>The importance of using a most comprehensive Knowledgebase for the identification of pathogenic variants
                 in cancer and inherited diseases.</i>
	      </span></td>
            </tr>

            <tr>
              <td><span ><p class="text-small"><br /><strong>12:35 - 14:00</strong></span></td>
              <td><span ><p class="text-small"><br /><b>Lunch Break and Poster Session with the Authors.</b>
              </span></td>
            </tr>
            <tr>
              <td colspan="2"><span  ><strong><br />
                  <br />
                    <p class="text-small">
                    Session 2<br>&nbsp;
<!--
    Genetic variants as effectors of change: disease and evolution. <br />
-->
              <br />
              </strong></span></td>
              </tr>

            <tr>
              <td><span ><p class="text-small"><strong>14:00 - 14:30</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small"><strong>Highlight Speaker:</strong>
              <a href="https://www2.unil.ch/cbg/index.php?title=Sven_Bergmann" target=blank><strong>Sven Bergmann.</strong></a> University of Lausanne,
	      Lausanne (Switzerland).
              <br>
               <i>
               Integration of molecular phenotypes into genome-wide association studies.</i>
              </span></td>
            </tr>
            <tr>
              <td><span ><p class="text-small"><strong>14:30 - 14:50</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small">
              <u>Cue Hyunkyu Lee.</u> ASAN Medical Center, Seoul (South Korea)
              <i><br>
		Increasing the power of meta-analysis of genome-wide association studies to detect heterogeneous effects.

              </i>
			</span></td>
            </tr>
            <tr>
              <td><span ><p class="text-small"><strong>14:50 - 15:10</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small">
              <u>Kyoko Watanabe.</u> Free University Amsterdam. Amsterdam (Netherlands) <br>
              <i>FUMA: Functional mapping and annotation of genetic associations.</i>
<span></td>
            </tr>
	    <tr>
              <td><span ><p class="text-small"><strong>15:10 - 15:30 </strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small">
              <u>Anastasia Gurinovich.</u> Boston University, Boston (MA).
              <br><i>PopCluster: A New Algorithm to Identify Genetic Variants with
		     Effects that Change with Ethnicity</i>
			  </span></td>
            </tr>
	   <tr>
              <td><span ><p class="text-small"><strong>15:30 - 15:50 </strong><br>&nbsp;</span></td>
              <td><span> <p class="text-small">
              <u>Magali Jaillard.</u> BioMérieux, Lyon (France).
              <br><i>
		Representing Genetic Determinants in Bacterial GWAS with
		Compacted De Bruijn Graphs.

              </i>
                          </span></td>
            </tr>
	     <tr>
              <td><span ><p class="text-small"><strong>15:50 - 16:05</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span ><p class="text-small"><b>Company Presentation:</b>
             <b><a href="https://www.variantyx.com/" target=blank>Alexander Kaplun - Variantyx.</a></b>
             <br>
              <i>One test to rule them all: Clinical grade Whole Genome Sequencing as first-line genetic test.</i>
              </span><br>&nbsp;</td>
            </tr>
            <tr>
            <td><span ><p class="text-small"><strong>16:05 - 16:30</strong><br>&nbsp;</span></td>
            <td><span ><p class="text-small"><b>Coffee Break</b></span><br>&nbsp;</td>
            </tr>
	    <tr>
              <td><span ><p class="text-small"><strong>16:30 - 17:00</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small"><strong>Highlight Speaker:</strong>
              <a href="https://www.bsse.ethz.ch/cbg/group/people/person-detail.html?persid=149417" target=blank><strong>Niko Beerenwinkel.</strong></a>
              Swiss Federal Institute of Technology (ETH), Zurich (Switzerland).<br>
              <i>
               Network-based integration of multi-omics data for prioritizing cancer genes.</i>
              </span></td>
            <tr>
              <td><span ><p class="text-small"><strong>17:00 - 17:20 </strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small">
              <u>Matteo Manica.</u> Swiss Federal Institute of Technology (ETH), Zurich (Switzerland).
              <br><i>
		Inferring clonal composition from multiple tumor biopsies.</i>
                          </span></td>
            </tr>
         <tr>
              <td><span ><p class="text-small"><strong>17:20 - 17:40</strong><br>&nbsp;</span></td>
              <td><span ><p class="text-small"><u>Sarah Sandmann.</u> Institute of Medical Informatics, Munster (Germany).
                          <br><i>
                         Evaluating Variant Calling Tools for Non-Matched Next
                         Generation Sequencing Data

                          </i>
              </span></td>
            </tr>
         <tr>
              <td><span ><p class="text-small"><strong>17:40 - 18:00</strong><br>&nbsp;<br>&nbsp;</span></td>
              <td><span ><p class="text-small"><u>Yoichiro Nakatani.</u> Trinity College Dublin, Dublin (Iteland).
                          <br><i>
			Genomes as documents of evolutionary history: a probabilistic macrosynteny
			model for the reconstruction of ancestral genomes.
                          </i>
              </span><br>&nbsp;</td>
            </tr>
              <tr>
                <td><span ><p class="text-small"><strong>18:00 - 18:10</strong></span></td>
                <td><span ><p class="text-small">Closing remarks from the committee.</span></td>
              </tr>
  						<!--
              <tr>
                <td><span  ><strong>18:05 - 20:00</strong></span></td>
                <td><span  ><b>Online Poster session with the authors.</b></span></td>
              </tr>
  					-->
          </tbody>
        </table>


    <br>
    <br>

    <div class='footer-ok'><div>

      </div>
    </div> <!-- /container -->


    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <script src="../assets/js/ie10-viewport-bug-workaround.js"></script>
    -->
  </body>
</html>
