<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <meta name="description" content="Variant Interpretation COSI">
    <meta name="author" content="Emidio Capriotti">
    <link rel="shortcut icon" href="../favicon.ico" type="image/vnd.microsoft.icon" />

    <title>Variant Interpretation - COSI </title>

    <!-- Bootstrap core CSS -->
    <link href="../css/bootstrap.min.css" rel="stylesheet">

    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <link href="../assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">
    -->

    <!-- Custom styles for this template -->
    <link href="../css/jumbotron-narrow-biofold.css" rel="stylesheet">

    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]
    <script src="../assets/js/ie-emulation-modes-warning.js"></script>
    -->



  </head>

  <body>
    <br>
    <div class="container">

      <div class="jumbotron">
        <table width="100%">
        <tr>
        <td width="80%">
	<div class='biofold' >

        <span style="color:#d97b2d;font-size:150%">VarI</span>
        <span style="color:#d97b2d;font-size:50%">c</span><span style="font-size:50%">ommunity </span>
        <span style="color:#d97b2d;font-size:50%">o</span><span style="font-size:50%">f  </span>
        <span style="color:#d97b2d;font-size:50%">s</span><span style="font-size:50%">pecial </span>
        <span style="color:#d97b2d;font-size:50%">i</span><span style="font-size:50%">nterest </span>

        </div>
        <!--
        <div class='biofold' ><span style="color:#c31e27;">Bio</span><span style="color:#4689a3;">molecules</span> <span style="color:#c31e27;">Fol</span><span style="color:#4689a3;">ding and </span><span style="color:#c31e27;">D</span><span style="color:#4689a3;">isease</span></div><br>
        -->
	</td>
	<td width="20%">
	<div class="right_cell">
        <a href="../index.html">
        <img src="../img/abacov.jpg" width="150px" align="center">
        </a>
      </div>
	</td>
	</tr>
	</table>
  <br>
	<table width="100%">
        <tr>
        <td>
        <div class="form-inline justify-content-center">
        <nav>
          <ul class="nav nav-pills">
            <li role="presentation"><a href="../index.html">Home</a></li>
            <li role="presentation" class='active'><a href="../schedule.html">Schedule</a></li>
            <li role="presentation"><a href="../submission.html">Submission</a></li>
            <li role="presentation"><a href="../publication.html">Publication</a></li>
            <li role="presentation"><a href="../company.html">Company</a></li>
	    <li role="presentation"><a href="../contact.html">Contact</a></li>
          </ul>
        </nav>
        </div>
        </td>
        </tr>
        </table>
    <br>
  	<br>

     <div class="text">
     <h4 class="title" >VarI COSI 2019</h4>
     </div>
     <br>
     <p class="text">
       <span style="color:#d97b2d;font-weight:400;">What:</span> Variant Interpretation Session ISMB/ECCB Conference 2019<br>
       <span style="color:#d97b2d;font-weight:400;">When:</span> 10:15 AM - 6:00 PM, July 24th, 2019<br>
       <span style="color:#d97b2d;font-weight:400;">Where:</span> Room Montreal (2nd Floor), Congress Center, Basel (Switzerland).
     </p>
      <table  border="0"  cellpadding="0" cellspacing="0" bgcolor="#FFFFFF" >
        <colgroup>
          <col style="width:15%">
          <col style="width:85%">
          </colgroup>
        <tbody>
          <tr>
            <td colspan="2"><span><strong><br />
                <br /><p class="text-small">
                  Session 1
            <br /><br />
            </strong></span></td>
            </tr>

              <tr>
                <td><span ><p class="text-small"><strong>10:15 - 10:20</strong><br>&nbsp;</span></td>
                <td><span ><p class="text-small"> Welcome from the committee<br>&nbsp;</span></td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>10:20 - 11:00</strong><br>&nbsp;</span></td>
                <td><span ><p class="text-small"><strong>Invited Speaker:</strong>
              <a href="http://grib.imim.es/about-us/laurai-furlong/" target=blank><strong>Laura Furlong.</strong></a>
              Hospital del Mar Medical Research Institute, Barcelona (Spain).
              <br>
              <i>
              The DisGeNET platform of disease genomics to support variant interpretation.
              </i>
                </span></td>
              </tr>


              <tr>
                <td><span ><p class="text-small"><strong>11:00 - 11:20</strong><br>&nbsp;<br></td>
                <td><p class="text-small">
                  <u>James Stephenson.</u> ENBL-EBI, Hinxton, (United Kingdom).  <br><i>
	                   Enrichment and Clustering of Rare Genetic Variants using Shared Protein Structure Domains.</i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>11:20 - 11:40</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Lambert Moyon.</u> Ecole Normale Superieure, Paris (France).
                  <br><i>
 	                    Annotation and prioritization of non-coding variants in the context of human diseases.
                    </i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>11:40 - 12:00</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Alexander Gress.</u>  Helmholtz Centre for Infection Research, Saarbrucken (Germany).
	                   <br><i>
		                 The importance of being unbiased: why protein structure and training setup are important for predicting
                novel pathogenic genetic variants.<br>
              </i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>12:00 - 12:20</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Alexandre Renaux.</u> Universite Libre de Bruxelles, Bruxelles (Belgium).
		                  <br>
		                      <i>
                Towards oligogenic disease prediction with ORVAL: a web-platform to uncover pathogenic variant combinations.
		              </i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>12:20 - 12:30</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Jun Cheng.</u> Technical University of Munich, Munich (Germany).
                  <br><i>
                    MMSplice: modular modeling improves the predictions of genetic variant effects on splicing.</i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>12:30 - 12:40</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Saikat Banerjee.</u> Max Planck Institute for Biophysical Chemistry, Gottingen (Germany).
              <br><i>
              Bayesian multiple logistic regression for case-control GWAS.
              </i>
                </td>
              </tr>


  						<tr>
  							<td><span ><p class="text-small"><br /><strong>12:40 - 14:00</strong></span></td>
  							<td><span ><p class="text-small"><br /><b>Lunch Break and <a href="https://iscb-ismb20.myconferencenow.com/presenters/?_poster_filter_by_track=vari-cosi" target=blank>
  								Online Posters.</a></b>
  							</span></td>
  						</tr>

  						<tr>
                <td colspan="2"><span><strong><br />
                    <br /><p class="text-small">

                      Session 2
  <!--
      Genetic variants as effectors of change: disease and evolution. <br />
  -->

                <br /><br />
                </strong></span></td>
                </tr>
  							<tr>
  	              <td><span ><p class="text-small"><strong>14:05 - 14:45</strong><br>&nbsp;</span></td>
  							<td><span><p class="text-small"><strong>Invited Speaker:</strong>
	      <a href="http://www.eva.mpg.de/genetics/staff/janet-kelso/"target=blank><strong>Janet Kelso.</strong></a>
              Max Planck Institute for Evolutionary Anthropology, Leipzig (Germany).
              <br>
               <i>
		              What ancient humans can teach us about modern genetic variation.
               </i>

                </span></td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>14:45 - 15:05</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Shaojie Zhang.</u> University of Central Florida, Orlando (FL).
                  <i><br>
	                 Efficient haplotype matching between a query and a panel for genealogical search.
                 </i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>15:05 - 15:25</strong><br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Daniel Carlin.</u> University of California, San Diego (CA).<br>
                    <i>
	                      A fast and flexible framework for network assisted genomic association.
	                   </i>
                </td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>15:25 - 15:45</strong><br>&nbsp;<br>&nbsp;</span></td>
                <td><p class="text-small">
                  <u>Erwin Frise</u> Fabric Genomics, Oakland (CA).
              <br><i>
	               An Artificial Intelligence Engine for High-Throughput ACMG/AMP Classification of Genetic Variants for Inherited
              Disease Clinical Gene Panels

              </i>
                </td>
              </tr>

  						<tr>
  		               <td ><span ><p class="text-small"><strong>15:45 - 16:00</strong><br>&nbsp;</span></td>
  		               <td ><span ><p class="text-small"><b>Company Presentation:</b>

                       <b><a href="https://www.variantyx.com/" target=blank>Alexander Kaplun - Variantyx.</a></b>
                       <br>
                       <i>Whole Genome Sequencing (WGS) as a first-line diagnostic test: Its success is in the details.
	                        </i>
  		               </span></td>
  		             </tr>
  								 <tr>
  		 							<td><span ><p class="text-small"><br /><strong>16:00 - 16:40</strong><br>&nbsp;</span></td>
  		 							<td><span ><p class="text-small"><br /><b>Coffee Break.<br>&nbsp;
  		 							</span></td>
  		 						</tr>
           <tr>
                <td><span ><p class="text-small"><strong>16:40 - 17:20</strong><br>&nbsp;</span></td>
                <td><span ><p class="text-small"><b>Invited Speaker:</b>
              <a href="https://www.mailman.columbia.edu/people/our-faculty/ii2135" target=blank>
             <strong>Iuliana Ionita-Laza.</strong></a> Columbia University, New York (NY).<br>
              <i>Integrative statistical approaches for predicting functional effects of variants in noncoding regions
                 of the<br>genome.
                </i>

                </span></td>
              </tr>
  	    <tr>
                <td><span ><p class="text-small"><strong>17:20 - 17:40</strong><br>&nbsp;</span></td>
  	      <td><span ><p class="text-small">

                <u>Alexander Schoenhuth.</u> Centrum Wiskunde en Informatica, Amsterdam (Netherland).
                   <br><i>
                    Using the structure of genome data in the design of deep neural networks for predicting amyotrophic lateral sclerosis from genotype.
                   </i>
                </span></td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>17:40 - 17:50</strong><br>&nbsp;</span></td>
                <td><span ><p class="text-small">
                  <u>Andrea Castro.</u> University of California, San Diego (CA).
                          <br><i>
			                         Elevated neoantigen levels in tumors with somatic mutations in the HLA-A, HLA-B, HLA-C and B2M genes.
                          </i>
  	              <span></td>
  			             </span></td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>17:50 - 18:00</strong><br>&nbsp;<br>&nbsp;</span></td>
                <td><span ><p class="text-small">

                  <u>Michal Sadowski.</u> University of Warsaw, Warsaw (Poland).
                            <br><i>
     			                      Spatial Chromatin Architecture Alteration by Structural Variations
                                in Human Genomes at Population Scale<br>&nbsp;
                            </i>
                <span></td>
              </tr>

              <tr>
                <td><span ><p class="text-small"><strong>18:00 - 18:05</strong></span></td>
                <td><span ><p class="text-small">Closing remarks from the committee.</span></td>
              </tr>
  						<!--
              <tr>
                <td><span class="textzone"><strong>18:05 - 20:00</strong></span></td>
                <td><span class="textzone"><b>Online Poster session with the authors.</b></span></td>
              </tr>
  					-->
          </tbody>
        </table>


    <br>
    <br>

    <div class='footer-ok'><div>

      </div>
    </div> <!-- /container -->


    <!-- IE10 viewport hack for Surface/desktop Windows 8 bug
    <script src="../assets/js/ie10-viewport-bug-workaround.js"></script>
    -->
  </body>
</html>
